<!DOCTYPE html><html lang="ru"><head><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"><title>Project Starter</title><meta name="description" content="Project template for static site (Gulp 4 + Webpack 4)"><meta name="keywords" content="gulp4, webpack4, es6, sass, pug, svg-icons"><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"><meta property="og:site_name" content="Project Starter"><meta property="og:url" content="https://github.com/Volosojui/project-starter"><meta property="og:title" content="Project Starter"><meta property="og:description" content="Project template for static site (Gulp 4 + Webpack 4)"><meta property="og:image" content="none"><meta name="twitter:card" content="summary"><meta name="twitter:title" content="Project Starter"><meta name="twitter:description" content="Project template for static site (Gulp 4 + Webpack 4)"><link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png"><link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png"><link rel="manifest" href="/site.webmanifest" crossorigin="use-credentials"><link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5"><meta name="msapplication-TileColor" content="#da532c"><meta name="theme-color" content="#ffffff"><link href="css/bundle.min.css" rel="stylesheet"></head><body><div class="svg-store"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true"><symbol id="icon-add-to-cart" viewbox="0 0 20 22">
  <path d="m 13.833008,0.57324225 c -0.5523,0 -1,0.44771895 -1,0.99999995 l 0,3.6152344 -3.4746098,0 c -0.56644,0 -1.0253906,0.4589406 -1.0253906,1.0253906 0,0.56644 0.4589507,1.0253906 1.0253906,1.0253906 l 3.4746098,0 0,3.6152352 c 0,0.5523 0.4477,1 1,1 0.5523,0 1,-0.4477 1,-1 l 0,-3.6152352 3.474609,0 c 0.5664,0 1.025391,-0.4589406 1.025391,-1.0253906 0,-0.56645 -0.458991,-1.0253906 -1.025391,-1.0253906 l -3.474609,0 0,-3.6152344 c 0,-0.552281 -0.4477,-0.99999995 -1,-0.99999995 z M 0.666992,6.3857422 l 0,1.7089845 1.6660156,0 3,6.4863283 -0.7421875,1.376953 1.9003907,0 0.091797,-0.171875 6.2089842,0 c 0.625,0 1.175684,-0.349006 1.458984,-0.878906 l 0.53711,-1 -1.904297,0 -0.0918,0.169922 -5.8496092,0 -1.9746094,-4.2734381 2.8652344,0 0,-1.7089843 -3.6582031,0 -0.7832031,-1.7089844 -2.7246094,0 z m 6,11.6230468 c -0.91667,0 -1.6582031,0.768885 -1.6582031,1.708985 0,0.9402 0.7415331,1.708984 1.6582031,1.708984 0.91667,0 1.6660156,-0.768784 1.6660156,-1.708984 0,-0.9401 -0.7493456,-1.708985 -1.6660156,-1.708985 z m 7.333984,0 c -0.9167,0 -1.658203,0.768885 -1.658203,1.708985 0,0.9402 0.741503,1.708984 1.658203,1.708984 0.9166,0 1.666016,-0.768784 1.666016,-1.708984 0,-0.9401 -0.749416,-1.708985 -1.666016,-1.708985 z"/>
</symbol><symbol id="icon-added-to-cart" viewbox="0 0 20 22">
  <path d="m 18.789513,1.2119172 c -0.2681,0 -0.535928,0.1039281 -0.736328,0.3144531 l -4.851563,5.0957035 c -0.0843,0.0886 -0.225293,0.08765 -0.308592,-0.002 l -1.552735,-1.667969 c -0.3957,-0.4254 -1.065403,-0.431825 -1.4707035,-0.01563 -0.3781,0.3882 -0.386678,1.00369 -0.01758,1.40039 l 3.0410165,3.267578 c 0.0746,0.0802 0.201744,0.08135 0.277343,0.002 l 6.355468,-6.677735 c 0.3738,-0.3927098 0.3738,-1.0096535 0,-1.4023435 C 19.32544,1.3158393 19.057611,1.2119112 18.789511,1.2119112 Z m -18.5957025,4.5332036 0,1.708984 1.6660156,0 3.0000004,6.4863282 -0.742188,1.376953 1.900391,0 0.0918,-0.171875 6.2089835,0 c 0.625,0 1.175684,-0.348906 1.458984,-0.878906 l 0.53711,-1 -1.904297,0 -0.0918,0.169922 -5.8496085,0 -1.97461,-4.2734382 2.865235,0 0,-1.708984 -3.658204,0 -0.783203,-1.708984 -2.724609,0 z m 6,11.6230462 c -0.9166,0 -1.658203,0.768785 -1.658203,1.708985 0,0.9402 0.741603,1.710937 1.658203,1.710937 0.9167,0 1.666016,-0.770737 1.666016,-1.710937 0,-0.9402 -0.749316,-1.708985 -1.666016,-1.708985 z m 7.3339835,0 c -0.916699,0 -1.658203,0.768785 -1.658203,1.708985 0,0.9402 0.741504,1.710937 1.658203,1.710937 0.9166,0 1.666016,-0.770737 1.666016,-1.710937 0,-0.9402 -0.749416,-1.708985 -1.666016,-1.708985 z"/>
</symbol><symbol id="icon-arrow-down" viewbox="0 0 12 12">
  <path d="M 1.53125,3.46875 0.46875,4.53125 6,10.060547 11.53125,4.53125 10.46875,3.46875 6,7.9375 1.53125,3.46875 Z"/>
</symbol><symbol id="icon-arrow-left" viewbox="0 0 12 12">
  <path d="M 7.46875 0.46875 L 1.9394531 6 L 7.46875 11.53125 L 8.53125 10.46875 L 4.0625 6 L 8.53125 1.53125 L 7.46875 0.46875 z "/>
</symbol><symbol id="icon-arrow-right" viewbox="0 0 12 12">
  <path d="M 4.53125,0.46875 3.46875,1.53125 7.9375,6 3.46875,10.46875 4.53125,11.53125 10.060547,6 4.53125,0.46875 Z"/>
</symbol><symbol id="icon-filters-mobile" viewbox="0 0 16 9">
<path fill-rule="evenodd" clip-rule="evenodd" d="M7.33966 6.39997C7.32184 6.51403 7.3126 6.63092 7.3126 6.74997C7.3126 7.05074 7.37161 7.33772 7.47869 7.59997L0.750097 7.59997C0.418726 7.59997 0.150097 7.33135 0.150097 6.99997C0.150097 6.6686 0.418726 6.39997 0.750097 6.39997L7.33966 6.39997ZM11.7855 6.39998L14.7501 6.39998C15.0815 6.39998 15.3501 6.6686 15.3501 6.99998C15.3501 7.33135 15.0815 7.59998 14.7501 7.59998L11.6465 7.59998C11.7536 7.33772 11.8126 7.05074 11.8126 6.74998C11.8126 6.63092 11.8034 6.51403 11.7855 6.39998Z" fill="black"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M8.16034 2.60003C8.17816 2.48597 8.1874 2.36908 8.1874 2.25003C8.1874 1.94926 8.12839 1.66228 8.02131 1.40003L14.7499 1.40003C15.0813 1.40003 15.3499 1.66866 15.3499 2.00003C15.3499 2.3314 15.0813 2.60003 14.7499 2.60003L8.16034 2.60003ZM3.71446 2.60003L0.749902 2.60002C0.418531 2.60002 0.149902 2.3314 0.149902 2.00002C0.149902 1.66865 0.418532 1.40002 0.749902 1.40002L3.85349 1.40003C3.74642 1.66228 3.6874 1.94926 3.6874 2.25002C3.6874 2.36908 3.69665 2.48597 3.71446 2.60003Z" fill="black"/>
<circle cx="9.5625" cy="6.75" r="1.65" transform="rotate(-180 9.5625 6.75)" stroke="black" stroke-width="1.2"/>
<circle cx="5.9375" cy="2.25" r="1.65" stroke="black" stroke-width="1.2"/>
</symbol><symbol id="icon-heart" viewbox="0 0 13 12">
<path d="M9.15582 0.75C8.01255 0.75 7.03564 1.5661 6.49891 2.12873C5.96218 1.5661 4.98745 0.75 3.84473 0.75C1.87509 0.75 0.5 2.12491 0.5 4.09304C0.5 6.26165 2.20782 7.66333 3.86 9.01912C4.64 9.65987 5.44727 10.3219 6.06636 11.0561C6.17055 11.179 6.32327 11.25 6.48364 11.25H6.51527C6.67618 11.25 6.82836 11.1784 6.932 11.0561C7.55218 10.3219 8.35891 9.65932 9.13945 9.01912C10.7911 7.66387 12.5 6.2622 12.5 4.09304C12.5 2.12491 11.1249 0.75 9.15582 0.75Z"/>
</symbol><symbol id="icon-list" viewbox="0 0 20 20">
  <path d="M 4 5 A 1.0001 1.0001 0 1 0 4 7 L 5 7 A 1.0001 1.0001 0 1 0 5 5 L 4 5 z M 8 5 L 8 7 L 16 7 L 16 5 L 8 5 z M 4 9 A 1.0001 1.0001 0 1 0 4 11 L 5 11 A 1.0001 1.0001 0 1 0 5 9 L 4 9 z M 8 9 L 8 11 L 16 11 L 16 9 L 8 9 z M 4 13 A 1.0001 1.0001 0 1 0 4 15 L 5 15 A 1.0001 1.0001 0 1 0 5 13 L 4 13 z M 8 13 L 8 15 L 16 15 L 16 13 L 8 13 z "/>
</symbol>
<symbol fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 482 433" id="icon-anim">
    <path d="M257.851 431.5H225m16.5-20.5v20M1 121h480v290H1z" stroke="#0F0E12" stroke-width="1.5"></path>
    <circle cx="93" cy="361" r="7" fill="#FF7740"></circle>
    <path d="M1.37 331.5H481" stroke="#0F0E12" stroke-width="1.5"></path>
    <circle cx="322" cy="365.548" r="31.5" transform="rotate(45 322 365.548)" fill="#69C7DC"></circle>
    <circle cx="322" cy="365.548" r="31.5" transform="rotate(45 322 365.548)" fill="url(#paint0_radial)" fill-opacity=".8"></circle>
    <circle cx="322" cy="365.548" r="31.5" transform="rotate(45 322 365.548)" fill="url(#paint1_radial)"></circle>
    <circle cx="322" cy="365.548" r="31.5" transform="rotate(45 322 365.548)" fill="url(#paint2_radial)"></circle>
    <circle cx="411.5" cy="312.5" r="17.5" fill="#FF7740"></circle>
    <circle cx="186" cy="306.154" r="39" transform="rotate(45 186 306.154)" fill="#E9BC93"></circle>
    <circle cx="186" cy="306.154" r="39" transform="rotate(45 186 306.154)" fill="url(#paint3_radial)" fill-opacity=".7"></circle>
    <circle cx="186" cy="306.154" r="39" transform="rotate(45 186 306.154)" fill="url(#paint4_radial)"></circle>
    <circle cx="186" cy="306.154" r="39" transform="rotate(45 186 306.154)" fill="url(#paint5_radial)"></circle>
    <circle r="8" transform="scale(1 -1) rotate(-45 -249.62 -250.918)" fill="#E8A4B4"></circle>
    <circle r="8" transform="scale(1 -1) rotate(-45 -249.62 -250.918)" fill="url(#paint6_radial)" fill-opacity=".9"></circle>
    <circle r="8" transform="scale(1 -1) rotate(-45 -249.62 -250.918)" fill="url(#paint7_radial)"></circle>
    <circle r="8" transform="scale(1 -1) rotate(-45 -249.62 -250.918)" fill="url(#paint8_radial)"></circle>
    <circle r="15" transform="scale(1 -1) rotate(-45 -105.32 -527.587)" fill="#E8A4B4"></circle>
    <circle r="15" transform="scale(1 -1) rotate(-45 -105.32 -527.587)" fill="url(#paint9_radial)" fill-opacity=".9"></circle>
    <circle r="15" transform="scale(1 -1) rotate(-45 -105.32 -527.587)" fill="url(#paint10_radial)"></circle>
    <circle r="15" transform="scale(1 -1) rotate(-45 -105.32 -527.587)" fill="url(#paint11_radial)"></circle>
    <circle cx="438" cy="207.021" transform="rotate(45 438 207.021)" fill="#69C7DC" r="8.5"></circle>
    <circle cx="438" cy="207.021" transform="rotate(45 438 207.021)" fill="url(#paint12_radial)" fill-opacity=".8" r="8.5"></circle>
    <circle cx="438" cy="207.021" transform="rotate(45 438 207.021)" fill="url(#paint13_radial)" r="8.5"></circle>
    <circle cx="438" cy="207.021" transform="rotate(45 438 207.021)" fill="url(#paint14_radial)" r="8.5"></circle>
    <circle cx="205" cy="196.556" r="11" transform="rotate(45 205 196.556)" fill="#E9BC93"></circle>
    <circle cx="205" cy="196.556" r="11" transform="rotate(45 205 196.556)" fill="url(#paint15_radial)" fill-opacity=".7"></circle>
    <circle cx="205" cy="196.556" r="11" transform="rotate(45 205 196.556)" fill="url(#paint16_radial)"></circle>
    <circle cx="205" cy="196.556" r="11" transform="rotate(45 205 196.556)" fill="url(#paint17_radial)"></circle>
    <circle cx="277" cy="156.607" r="7.5" transform="rotate(45 277 156.607)" fill="#E9BC93"></circle>
    <circle cx="277" cy="156.607" r="7.5" transform="rotate(45 277 156.607)" fill="url(#paint18_radial)" fill-opacity=".7"></circle>
    <circle cx="277" cy="156.607" r="7.5" transform="rotate(45 277 156.607)" fill="url(#paint19_radial)"></circle>
    <circle cx="277" cy="156.607" r="7.5" transform="rotate(45 277 156.607)" fill="url(#paint20_radial)"></circle>
    <circle r="39" transform="scale(1 -1) rotate(-45 -173.13 -162.441)" fill="#E8A4B4"></circle>
    <circle r="39" transform="scale(1 -1) rotate(-45 -173.13 -162.441)" fill="url(#paint21_radial)" fill-opacity=".9"></circle>
    <circle r="39" transform="scale(1 -1) rotate(-45 -173.13 -162.441)" fill="url(#paint22_radial)"></circle>
    <circle r="39" transform="scale(1 -1) rotate(-45 -173.13 -162.441)" fill="url(#paint23_radial)"></circle>
    <circle cx="185.5" cy="83.5" r="17.5" fill="#FF7740"></circle>
    <circle cx="364" cy="101.154" r="39" transform="rotate(45 364 101.154)" fill="#69C7DC"></circle>
    <circle cx="364" cy="101.154" r="39" transform="rotate(45 364 101.154)" fill="url(#paint24_radial)" fill-opacity=".8"></circle>
    <circle cx="364" cy="101.154" r="39" transform="rotate(45 364 101.154)" fill="url(#paint25_radial)"></circle>
    <circle cx="364" cy="101.154" r="39" transform="rotate(45 364 101.154)" fill="url(#paint26_radial)"></circle>
    <path d="M241.5 1.54V121M257.851 1.5H225" stroke="#0F0E12" stroke-width="1.5"></path>
    <defs>
      <radialGradient id="paint0_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(43.875 0 0 49.6742 299.5 365.548)">
        <stop stop-color="#fff"></stop>
        <stop offset="1" stop-color="#fff" stop-opacity="0"></stop>
      </radialGradient>
      <radialGradient id="paint1_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(297.25 365.548) scale(56.25)">
        <stop offset=".682" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12" stop-opacity=".19"></stop>
      </radialGradient>
      <radialGradient id="paint2_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(306.25 365.548) scale(159.75)">
        <stop stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12"></stop>
      </radialGradient>
      <radialGradient id="paint3_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(54.3214 0 0 61.5015 158.143 306.154)">
        <stop stop-color="#fff"></stop>
        <stop offset="1" stop-color="#fff" stop-opacity="0"></stop>
      </radialGradient>
      <radialGradient id="paint4_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="rotate(-30 698.591 -105.001) scale(125.357)">
        <stop offset=".474" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12" stop-opacity=".19"></stop>
      </radialGradient>
      <radialGradient id="paint5_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(146.652 306.154) scale(254.893)">
        <stop offset=".168" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12"></stop>
      </radialGradient>
      <radialGradient id="paint6_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(11.1429 0 0 12.6157 2.286 8)">
        <stop stop-color="#fff" stop-opacity=".9"></stop>
        <stop offset="1" stop-color="#fff" stop-opacity="0"></stop>
      </radialGradient>
      <radialGradient id="paint7_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(1.714 8) scale(14.2857)">
        <stop offset=".682" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12" stop-opacity=".19"></stop>
      </radialGradient>
      <radialGradient id="paint8_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(4 8) scale(40.5714)">
        <stop offset=".093" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12"></stop>
      </radialGradient>
      <radialGradient id="paint9_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(20.8929 0 0 23.6544 4.286 15)">
        <stop stop-color="#fff" stop-opacity=".9"></stop>
        <stop offset="1" stop-color="#fff" stop-opacity="0"></stop>
      </radialGradient>
      <radialGradient id="paint10_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(3.214 15) scale(26.7857)">
        <stop offset=".682" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12" stop-opacity=".19"></stop>
      </radialGradient>
      <radialGradient id="paint11_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(7.5 15) scale(76.0714)">
        <stop offset=".093" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12"></stop>
      </radialGradient>
      <radialGradient id="paint12_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(11.8393 0 0 13.4042 431.929 207.021)">
        <stop stop-color="#fff"></stop>
        <stop offset="1" stop-color="#fff" stop-opacity="0"></stop>
      </radialGradient>
      <radialGradient id="paint13_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(431.321 207.021) scale(15.1786)">
        <stop offset=".682" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12" stop-opacity=".19"></stop>
      </radialGradient>
      <radialGradient id="paint14_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(433.75 207.021) scale(43.1071)">
        <stop stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12"></stop>
      </radialGradient>
      <radialGradient id="paint15_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(15.3214 0 0 17.3466 197.143 196.556)">
        <stop stop-color="#fff"></stop>
        <stop offset="1" stop-color="#fff" stop-opacity="0"></stop>
      </radialGradient>
      <radialGradient id="paint16_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="rotate(-30 478.954 -259.153) scale(35.3571)">
        <stop offset=".474" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12" stop-opacity=".19"></stop>
      </radialGradient>
      <radialGradient id="paint17_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(193.902 196.556) scale(71.8929)">
        <stop offset=".168" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12"></stop>
      </radialGradient>
      <radialGradient id="paint18_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(10.4464 0 0 11.8272 271.643 156.607)">
        <stop stop-color="#fff"></stop>
        <stop offset="1" stop-color="#fff" stop-opacity="0"></stop>
      </radialGradient>
      <radialGradient id="paint19_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="rotate(-30 437.329 -421.47) scale(24.1071)">
        <stop offset=".474" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12" stop-opacity=".19"></stop>
      </radialGradient>
      <radialGradient id="paint20_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(269.433 156.607) scale(49.0179)">
        <stop offset=".168" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12"></stop>
      </radialGradient>
      <radialGradient id="paint21_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(54.3214 0 0 61.5014 11.143 39)">
        <stop stop-color="#fff" stop-opacity=".9"></stop>
        <stop offset="1" stop-color="#fff" stop-opacity="0"></stop>
      </radialGradient>
      <radialGradient id="paint22_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(8.357 39) scale(69.6429)">
        <stop offset=".682" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12" stop-opacity=".19"></stop>
      </radialGradient>
      <radialGradient id="paint23_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(19.5 39) scale(197.786)">
        <stop offset=".093" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12"></stop>
      </radialGradient>
      <radialGradient id="paint24_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(54.3214 0 0 61.5015 336.143 101.154)">
        <stop stop-color="#fff"></stop>
        <stop offset="1" stop-color="#fff" stop-opacity="0"></stop>
      </radialGradient>
      <radialGradient id="paint25_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(333.357 101.154) scale(69.6429)">
        <stop offset=".682" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12" stop-opacity=".19"></stop>
      </radialGradient>
      <radialGradient id="paint26_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(344.5 101.154) scale(197.786)">
        <stop stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12"></stop>
      </radialGradient>
    </defs>
  </symbol>
  <symbol fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 14 16" id="icon-file-btn">
    <path clip-rule="evenodd" d="M7.749.5h-5.25A1.5 1.5 0 00.999 2v12a1.5 1.5 0 001.5 1.5h9a1.5 1.5 0 001.5-1.5V5.75L7.748.5z" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round"></path>
    <path d="M7.751.5v5.25h5.25" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round"></path>
  </symbol>
  <symbol fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 17 16" id="icon-sidebar">
    <rect x="15.875" y="1" width="14" height="14" rx="2" transform="rotate(90 15.875 1)" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round"></rect>
    <path d="M15.125 5.666h-14" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round"></path>
  </symbol>
  <symbol fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 264 295" id="icon-japan">
    <path d="M30.593 288.982L21.283 293l-4.654-13.394v6.697H9.98V268.89l4.655-19.422L2 260.853l2.66-16.743 16.624-10.716 9.31-7.366 21.278-18.753 24.604-5.358 13.964 3.349h11.969v-8.037l16.624-14.064 19.949-10.715 22.609-22.771 4.655-24.78v-23.44l8.644-14.065h12.634v14.065l9.31 12.055v22.77l-9.31 13.395-5.319 6.697 2.659 14.064-2.659 20.092v11.385h-7.315v9.377l-8.644 4.018 1.994-4.018v-4.019l-6.649 2.009-8.645 6.028h-4.654l-9.31 10.716h-15.294v-4.019l-8.645-6.697-2.659 10.716 6.649 3.348-11.304 6.028-1.995 6.027-9.974 8.037-5.32-16.074 5.32-11.385h-5.32l-7.98 13.395v6.027l-7.98 8.037-10.639-5.358-11.969 16.743-7.315-14.064 7.315-11.385h14.63l3.324-6.028h9.975l7.314-7.367-20.614 4.019-21.944 9.376H25.274l1.994 8.706h7.98l-7.98 5.358h7.98l4.655 7.367-6.65 26.119h-5.985l3.325 6.028z" stroke="#000" stroke-width="1.5"></path>
    <path d="M131.129 208.964l-.217-40.205" stroke="#E9BC93" stroke-width="1.5"></path>
    <ellipse rx="14.033" ry="13.996" transform="matrix(-.61174 .79051 -.78694 -.61773 131.353 156.144)" fill="#E9BC93"></ellipse>
    <ellipse rx="14.033" ry="13.996" transform="matrix(-.61174 .79051 -.78694 -.61773 131.353 156.144)" fill="url(#paint0_radial)" fill-opacity=".7"></ellipse>
    <ellipse rx="14.033" ry="13.996" transform="matrix(-.61174 .79051 -.78694 -.61773 131.353 156.144)" fill="url(#paint1_radial)"></ellipse>
    <ellipse rx="14.033" ry="13.996" transform="matrix(-.61174 .79051 -.78694 -.61773 131.353 156.144)" fill="url(#paint2_radial)"></ellipse>
    <path d="M177.55 78.018l-8.645 9.376-4.654-9.376 4.654-3.348-4.654-5.358v-8.037l15.959-12.724h9.309l7.315-12.056V17.073l-3.99-13.394L202.818 1l18.619 20.092 20.614 10.715 7.98-10.715h3.989l-3.989 12.724L262 44.532l-17.289 8.707-17.954 8.036-5.32 13.395-26.598-16.743-11.305 5.357-7.979-2.009 1.995 10.046 8.644 6.697h-8.644z" stroke="#000" stroke-width="1.5"></path>
    <defs>
      <radialGradient id="paint0_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(19.5453 0 0 22.0712 4.01 13.996)">
        <stop stop-color="#fff"></stop>
        <stop offset="1" stop-color="#fff" stop-opacity="0"></stop>
      </radialGradient>
      <radialGradient id="paint1_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(39.06174 -22.49362 22.46428 39.01079 -.326 24.43)">
        <stop offset=".474" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12" stop-opacity=".19"></stop>
      </radialGradient>
      <radialGradient id="paint2_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(91.7128 0 0 91.474 -.125 13.996)">
        <stop offset=".168" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12"></stop>
      </radialGradient>
    </defs>
  </symbol>
  <symbol fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 343 189" id="icon-russia">
    <path d="M29.332 178.073c-2.108 2.693.879 7.406 2.636 9.426h-3.294l-2.635-2.692-7.907-6.734-14.496-12.118L1 158.549l2.636-3.366 7.248-9.426 7.248-2.694-2.636-5.386-6.589-6.059v-6.06l1.977-6.733-5.93-2.693 1.976-4.04v-7.406l-1.318-2.693-.659-2.693 5.93-12.793 7.249-6.733 9.883-.673 3.954-7.406v-4.713l5.271-4.04v-4.04l6.59-4.712.658-4.04 5.93-1.347 2.636 2.02 4.613 3.367 5.93 7.406-7.248 2.693-9.884-4.04v4.04l-2.636 6.06v3.366l8.566-2.02.66-2.02 11.2-2.02 7.908-.673 1.977-1.347 4.612-1.346 5.93 1.346 7.907-4.04 13.178-1.346 8.566 6.733V60.25l5.272-13.466h5.271l-3.954 8.08 3.954 5.386-5.271 9.426-8.566 4.04 6.589 6.06 8.566-6.06-1.318-4.04v-2.693l3.295 3.367 7.248-.674-1.318-4.04-5.93-1.346-1.977-6.06 3.294-11.446v8.08h4.613l-1.977-2.693 4.612-1.347 1.977 6.06 3.954 1.346 3.735-4.713-7.689-2.693V44.09l7.689-1.346 6.807-4.713 10.543-6.733 1.318-5.386 3.953 1.346h6.589l5.931 5.387-5.931 12.792 13.179-7.406h8.566l7.907-1.347 5.93 1.347 5.93 6.06 1.977-4.04h6.589l3.294-4.713-1.317-6.06 8.566-.999-1.318 4.366 7.907-4.366 8.566.999 5.93-3.366 9.884 1.346 9.224-5.386 5.272-1.347 5.93-1.346-5.93-2.693 5.93-3.367 23.721-8.08h9.225V2.348L330.457 1v4.04l3.954 2.02-5.93 2.02h-3.295l-4.612 2.02h-4.613v3.366l3.295 2.02-2.636 4.04h3.954l4.612 1.346 2.636-1.347 4.612 2.02-3.953 3.367-.659 5.386 1.318 1.347v9.426h-5.272v6.06l-3.294-2.694v3.366l-1.318 3.367 4.612 6.733h3.954l1.318 1.346 1.317 2.02 1.977-1.346 1.977 3.366-1.977-1.347-1.318 1.347 2.636 2.693 3.953 2.693-1.976.674v3.366l1.976 3.366v4.713l1.977 2.694 1.318 6.06-10.543-7.407-8.566-6.06-4.612-6.06-1.977-4.712.659-2.02-.659-6.06V48.13l-7.248-6.06h-1.318l1.318 10.1-3.953-4.04-5.93 6.733 1.976 6.733h6.589c0 1.795-1.317 5.386-6.589 5.386h-7.248v4.04l-5.271 2.02-9.225 6.733V93.24l-3.294 11.446h6.589l5.93-2.02h7.907l4.612 8.753 6.589 14.139 1.318 14.812-7.907 20.199-1.976 3.366-2.636-6.732-2.636-7.407 5.272-2.02-2.636-16.543-2.636 4.424h-9.225l-1.976-2.693h-12.52l-11.86-12.792-11.861 4.039v3.367l2.636 5.386-2.636 12.119h-10.542l-5.931 4.713-9.883 2.694-7.907-2.694-5.93 2.694-5.931-2.694v-4.713h-9.224l-4.613 2.693 4.613 6.733-9.884 2.694v-2.694l-7.248-2.019-7.907 4.713-7.907 6.732-9.884-9.426h-13.178v-6.733l-4.613-7.406-4.612 2.693-5.271-2.693-7.248-2.693v-4.713l-4.613-3.366L86 132.964v4.713l-5.271 7.406 1.976 7.407-16.472-7.407h-3.954v-2.02l-8.566-2.693-9.225 7.406-5.93-4.713-3.953 9.427 3.953 11.445-1.977 6.058c-1.537-1.122-5.14-2.693-7.249 0-2.635 3.367 2.636 4.713 0 8.08z" stroke="#0F0E12" stroke-width="1.5"></path>
    <path d="M54.372 106.045v-40.42" stroke="#69C7DC" stroke-width="1.5"></path>
    <ellipse rx="13.947" ry="14.031" transform="matrix(.7927 .6096 -.59304 .80518 53.813 53.601)" fill="#69C7DC"></ellipse>
    <ellipse rx="13.947" ry="14.031" transform="matrix(.7927 .6096 -.59304 .80518 53.813 53.601)" fill="url(#paint0_radial)" fill-opacity=".7"></ellipse>
    <ellipse rx="13.947" ry="14.031" transform="matrix(.7927 .6096 -.59304 .80518 53.813 53.601)" fill="url(#paint1_radial)"></ellipse>
    <ellipse rx="13.947" ry="14.031" transform="matrix(.7927 .6096 -.59304 .80518 53.813 53.601)" fill="url(#paint2_radial)"></ellipse>
    <defs>
      <radialGradient id="paint0_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(19.4263 0 0 22.1258 3.985 14.03)">
        <stop stop-color="#fff"></stop>
        <stop offset="1" stop-color="#fff" stop-opacity="0"></stop>
      </radialGradient>
      <radialGradient id="paint1_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(24.9055 0 0 25.0547 2.989 14.03)">
        <stop offset=".682" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12" stop-opacity=".19"></stop>
      </radialGradient>
      <radialGradient id="paint2_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(70.7317 0 0 71.1554 6.974 14.03)">
        <stop stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12"></stop>
      </radialGradient>
    </defs>
  </symbol>
  <symbol fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 310 189" id="icon-usa">
    <path d="M41.015 128.957L26.583 125l-3.28-9.234-11.151-7.915-1.312-5.277L6.904 94 4.28 83.447 1 63.66l.656-12.532 4.592-14.511 7.216-17.149V2.32l7.871 5.936v5.936l1.968-3.297V5.617L24.615 1l80.03 14.51 53.791 3.298 13.12 3.958 14.431 3.298-5.904 7.255 3.936 1.979h9.184l9.84-1.979 15.743.66 1.312 2.638h-11.151l-8.528 7.255v12.532l.656 13.851h5.904l-.656-21.106 5.904-7.915 9.839 4.617-4.592 9.894 3.936 1.319v-3.958l2.624-1.319 2.624 6.596-2.624 9.894 6.56 3.297 15.744-9.893v-5.936h5.903l6.56-3.958V47.17l6.56-7.255 16.4-3.298 5.248-5.936v-3.298l3.279-12.532 6.56-.66 5.248 11.873L308 30.02l-6.56 3.958h-3.28v3.957l-5.248 3.298-1.312 7.915-.656 2.638 3.28 6.596-18.367 9.894-.656 1.978 3.28 3.298-4.592 8.575-1.312 7.915 1.312 5.276-1.968 1.32-1.968-3.958-1.312-9.234V77.51l-5.248 3.297 2.624 7.256v4.617l3.936 4.617 2.624 5.936-.656 9.234-5.904 4.617-2.624 4.617-4.592 1.319-1.312 4.617-7.871 7.256-4.592 10.553 5.248 13.851 7.215 12.532v9.893L252.897 187l-4.591-6.596-7.872-8.574 1.312-2.639-.656-1.319-2.624 2.639v-9.234l-9.184-7.256-6.56 2.639-7.216-3.958-5.247.66h-4.592l-1.312-1.979-4.592 1.979h-8.528l-2.624 1.978 7.216 1.979 2.624 5.277v1.978l-6.56-4.617-1.968 4.617-9.184-4.617H167.62l-21.648 14.511V187l-7.216-1.979-4.591-4.617v-3.957h-3.28l-9.184-16.49h-8.528l-3.936 4.617-7.872-4.617-2.623-7.255-4.592-7.255-11.808-5.936v1.978H68.566l-27.551-12.532z" stroke="#000" stroke-width="1.5"></path>
    <path d="M30.519 113.799V74.202" stroke="#D09DAA" stroke-width="1.5"></path>
    <ellipse rx="13.822" ry="13.805" transform="matrix(.61716 .78683 -.78355 .62133 30.201 62.666)" fill="#E8A4B4"></ellipse>
    <ellipse rx="13.822" ry="13.805" transform="matrix(.61716 .78683 -.78355 .62133 30.201 62.666)" fill="url(#paint0_radial)" fill-opacity=".9"></ellipse>
    <ellipse rx="13.822" ry="13.805" transform="matrix(.61716 .78683 -.78355 .62133 30.201 62.666)" fill="url(#paint1_radial)"></ellipse>
    <ellipse rx="13.822" ry="13.805" transform="matrix(.61716 .78683 -.78355 .62133 30.201 62.666)" fill="url(#paint2_radial)"></ellipse>
    <defs>
      <radialGradient id="paint0_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(19.2523 0 0 21.7693 3.95 13.805)">
        <stop stop-color="#fff" stop-opacity=".9"></stop>
        <stop offset="1" stop-color="#fff" stop-opacity="0"></stop>
      </radialGradient>
      <radialGradient id="paint1_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(24.6825 0 0 24.6511 2.962 13.805)">
        <stop offset=".682" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12" stop-opacity=".19"></stop>
      </radialGradient>
      <radialGradient id="paint2_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(70.0982 0 0 70.0091 6.911 13.805)">
        <stop offset=".093" stop-color="#0F0E12" stop-opacity="0"></stop>
        <stop offset="1" stop-color="#0F0E12"></stop>
      </radialGradient>
    </defs>
  </symbol></svg>
</div><div class="layout"><header class="header" id="top"><div class="header__container container"><a class="header__logo" href="./home.html"><svg width="207" height="43" viewBox="0 0 207 43" fill="none" xmlns="http://www.w3.org/2000/svg">
<path fill-rule="evenodd" clip-rule="evenodd" d="M0.569336 17.6421V0.322998H5.89438L9.91829 12.2329H9.96648L13.7735 0.322998H19.0985V17.6421H15.5565V5.36833H15.5083L11.2917 17.6421H8.37619L4.15952 5.48961H4.11133V17.6421H0.569336ZM22.4292 17.6421V0.322998H26.2121V17.6421H22.4292ZM0.569336 24.5794V41.8985H12.7374V38.6967H4.35229V24.5794H0.569336ZM21.4303 28.8486L19.1894 35.2038H23.6471L21.4785 28.8486H21.4303ZM13.0452 41.8985L19.5509 24.5794H23.4302L29.8636 41.8985H25.9361L24.635 38.0417H18.2015L16.8522 41.8985H13.0452ZM35.1302 31.5895V27.5387H38.6722C39.0095 27.5387 39.3308 27.571 39.636 27.6358C39.9572 27.6843 40.2384 27.7813 40.4793 27.9268C40.7203 28.0724 40.913 28.2745 41.0576 28.5332C41.2022 28.7919 41.2745 29.1234 41.2745 29.5278C41.2745 30.2554 41.0576 30.7809 40.6239 31.1044C40.1902 31.4278 39.636 31.5895 38.9613 31.5895H35.1302ZM31.3472 24.5794V41.8985H39.6842C40.4552 41.8985 41.2022 41.8015 41.925 41.6074C42.664 41.4134 43.3146 41.1142 43.8768 40.71C44.455 40.3056 44.9129 39.7882 45.2502 39.1575C45.5875 38.5107 45.7562 37.7507 45.7562 36.8774C45.7562 35.794 45.4911 34.8722 44.9611 34.1122C44.4471 33.336 43.6599 32.7942 42.5997 32.487C43.3708 32.115 43.949 31.6381 44.3346 31.0559C44.7362 30.4737 44.937 29.7461 44.937 28.8728C44.937 28.0642 44.8004 27.385 44.5273 26.8353C44.2703 26.2854 43.9009 25.8488 43.419 25.5254C42.937 25.1859 42.3588 24.9433 41.6841 24.7977C41.0094 24.6522 40.2625 24.5794 39.4432 24.5794H31.3472ZM35.1302 38.9392V34.185H39.2505C40.0697 34.185 40.7283 34.379 41.2263 34.7671C41.7243 35.139 41.9732 35.7697 41.9732 36.6591C41.9732 37.1118 41.893 37.4839 41.7323 37.7749C41.5877 38.066 41.387 38.3004 41.1299 38.4784C40.8729 38.64 40.5757 38.7613 40.2384 38.8422C39.9011 38.9068 39.5476 38.9392 39.1782 38.9392H35.1302ZM51.5896 31.1044C51.4291 31.8159 51.3487 32.5516 51.3487 33.3117C51.3487 34.0394 51.4291 34.7509 51.5896 35.4463C51.7664 36.1255 52.0395 36.74 52.4089 37.2898C52.7784 37.8395 53.2603 38.2843 53.8546 38.6239C54.449 38.9473 55.1718 39.109 56.0232 39.109C56.8746 39.109 57.5975 38.9473 58.1917 38.6239C58.7861 38.2843 59.268 37.8395 59.6375 37.2898C60.007 36.74 60.272 36.1255 60.4326 35.4463C60.6094 34.7509 60.6976 34.0394 60.6976 33.3117C60.6976 32.5516 60.6094 31.8159 60.4326 31.1044C60.272 30.3928 60.007 29.7622 59.6375 29.2124C59.268 28.6464 58.7861 28.2017 58.1917 27.8783C57.5975 27.5387 56.8746 27.3689 56.0232 27.3689C55.1718 27.3689 54.449 27.5387 53.8546 27.8783C53.2603 28.2017 52.7784 28.6464 52.4089 29.2124C52.0395 29.7622 51.7664 30.3928 51.5896 31.1044ZM48.144 36.8532C47.7585 35.7535 47.5657 34.5731 47.5657 33.3117C47.5657 32.018 47.7585 30.8214 48.144 29.7218C48.5296 28.606 49.0837 27.6357 49.8066 26.811C50.5295 25.9863 51.413 25.3394 52.4571 24.8705C53.5173 24.4015 54.706 24.1671 56.0232 24.1671C57.3565 24.1671 58.5452 24.4015 59.5893 24.8705C60.6335 25.3394 61.5169 25.9863 62.2397 26.811C62.9626 27.6357 63.5168 28.606 63.9023 29.7218C64.2878 30.8214 64.4806 32.018 64.4806 33.3117C64.4806 34.5731 64.2878 35.7535 63.9023 36.8532C63.5168 37.9366 62.9626 38.8826 62.2397 39.6912C61.5169 40.4997 60.6335 41.1384 59.5893 41.6074C58.5452 42.0602 57.3565 42.2866 56.0232 42.2866C54.706 42.2866 53.5173 42.0602 52.4571 41.6074C51.413 41.1384 50.5295 40.4997 49.8066 39.6912C49.0837 38.8826 48.5296 37.9366 48.144 36.8532ZM70.8495 32.4143V27.5387H74.9939C75.8613 27.5387 76.5119 27.7328 76.9456 28.1209C77.3793 28.4928 77.5962 29.0992 77.5962 29.9401C77.5962 30.8133 77.3793 31.444 76.9456 31.8321C76.5119 32.2202 75.8613 32.4143 74.9939 32.4143H70.8495ZM67.0666 24.5794V41.8985H70.8495V35.131H74.6325C75.5802 35.131 76.2628 35.3411 76.6806 35.7617C77.0982 36.1821 77.3712 36.845 77.4998 37.7507C77.5962 38.4459 77.6685 39.1736 77.7167 39.9338C77.7648 40.6937 77.8933 41.3487 78.1022 41.8985H81.8851C81.7085 41.656 81.5719 41.3649 81.4755 41.0253C81.3951 40.6695 81.3309 40.2976 81.2828 39.9095C81.2506 39.5214 81.2265 39.1413 81.2105 38.7694C81.1944 38.3974 81.1783 38.0741 81.1623 37.7992C81.1301 37.3626 81.0659 36.926 80.9695 36.4893C80.8891 36.0527 80.7527 35.6565 80.5599 35.3008C80.3671 34.9288 80.1181 34.6134 79.8129 34.3548C79.5077 34.0798 79.1222 33.8777 78.6564 33.7484V33.6999C79.6202 33.3118 80.3108 32.7457 80.7286 32.0019C81.1623 31.258 81.3791 30.3767 81.3791 29.358C81.3791 28.6949 81.2587 28.0804 81.0177 27.5145C80.7927 26.9323 80.4554 26.4229 80.0057 25.9863C79.572 25.5497 79.0419 25.2101 78.4154 24.9675C77.8049 24.7088 77.1143 24.5794 76.3432 24.5794H67.0666ZM91.0336 28.8486L88.7927 35.2038H93.2503L91.0817 28.8486H91.0336ZM82.6484 41.8985L89.1541 24.5794H93.0335L99.4669 41.8985H95.5394L94.2382 38.0417H87.8048L86.4555 41.8985H82.6484ZM104.758 27.7813V41.8985H108.54V27.7813H113.697V24.5794H99.6012V27.7813H104.758ZM118.957 31.1044C118.796 31.8159 118.716 32.5516 118.716 33.3117C118.716 34.0394 118.796 34.7509 118.957 35.4463C119.134 36.1255 119.408 36.74 119.777 37.2898C120.146 37.8395 120.628 38.2843 121.222 38.6239C121.817 38.9473 122.54 39.109 123.391 39.109C124.242 39.109 124.965 38.9473 125.56 38.6239C126.154 38.2843 126.636 37.8395 127.005 37.2898C127.374 36.74 127.639 36.1255 127.8 35.4463C127.977 34.7509 128.065 34.0394 128.065 33.3117C128.065 32.5516 127.977 31.8159 127.8 31.1044C127.639 30.3928 127.374 29.7622 127.005 29.2124C126.636 28.6464 126.154 28.2017 125.56 27.8783C124.965 27.5387 124.242 27.3689 123.391 27.3689C122.54 27.3689 121.817 27.5387 121.222 27.8783C120.628 28.2017 120.146 28.6464 119.777 29.2124C119.408 29.7622 119.134 30.3928 118.957 31.1044ZM115.512 36.8532C115.126 35.7535 114.934 34.5731 114.934 33.3117C114.934 32.018 115.126 30.8214 115.512 29.7218C115.897 28.606 116.451 27.6357 117.174 26.811C117.897 25.9863 118.781 25.3394 119.825 24.8705C120.885 24.4015 122.073 24.1671 123.391 24.1671C124.724 24.1671 125.913 24.4015 126.957 24.8705C128.001 25.3394 128.885 25.9863 129.607 26.811C130.33 27.6357 130.885 28.606 131.27 29.7218C131.656 30.8214 131.848 32.018 131.848 33.3117C131.848 34.5731 131.656 35.7535 131.27 36.8532C130.885 37.9366 130.33 38.8826 129.607 39.6912C128.885 40.4997 128.001 41.1384 126.957 41.6074C125.913 42.0602 124.724 42.2866 123.391 42.2866C122.073 42.2866 120.885 42.0602 119.825 41.6074C118.781 41.1384 117.897 40.4997 117.174 39.6912C116.451 38.8826 115.897 37.9366 115.512 36.8532ZM138.217 32.4143V27.5387H142.362C143.229 27.5387 143.88 27.7328 144.313 28.1209C144.747 28.4928 144.964 29.0992 144.964 29.9401C144.964 30.8133 144.747 31.444 144.313 31.8321C143.88 32.2202 143.229 32.4143 142.362 32.4143H138.217ZM134.434 24.5794V41.8985H138.217V35.131H142C142.947 35.131 143.631 35.3411 144.048 35.7617C144.465 36.1821 144.739 36.845 144.867 37.7507C144.964 38.4459 145.036 39.1736 145.084 39.9338C145.133 40.6937 145.261 41.3487 145.47 41.8985H149.253C149.076 41.656 148.94 41.3649 148.843 41.0253C148.763 40.6695 148.699 40.2976 148.65 39.9095C148.619 39.5214 148.595 39.1413 148.578 38.7694C148.562 38.3974 148.546 38.0741 148.53 37.7992C148.498 37.3626 148.434 36.926 148.337 36.4893C148.257 36.0527 148.12 35.6565 147.928 35.3008C147.735 34.9288 147.486 34.6134 147.181 34.3548C146.875 34.0798 146.49 33.8777 146.024 33.7484V33.6999C146.988 33.3118 147.679 32.7457 148.096 32.0019C148.53 31.258 148.747 30.3767 148.747 29.358C148.747 28.6949 148.626 28.0804 148.385 27.5145C148.161 26.9323 147.824 26.4229 147.373 25.9863C146.94 25.5497 146.41 25.2101 145.783 24.9675C145.173 24.7088 144.482 24.5794 143.711 24.5794H134.434ZM151.823 41.8985V24.5794H155.606V41.8985H151.823ZM158.929 24.5794V41.8985H171.989V38.6967H162.712V34.4518H171.049V31.4925H162.712V27.7813H171.796V24.5794H158.929ZM173.47 36.1497H177.132C177.132 36.7319 177.236 37.2251 177.445 37.6294C177.654 38.0336 177.927 38.3651 178.265 38.6239C178.618 38.8665 179.028 39.0524 179.493 39.1818C179.959 39.2949 180.441 39.3516 180.939 39.3516C181.276 39.3516 181.638 39.3273 182.023 39.2788C182.409 39.2141 182.77 39.1009 183.108 38.9392C183.445 38.7775 183.727 38.5592 183.951 38.2843C184.176 37.9932 184.288 37.6294 184.288 37.1928C184.288 36.7237 184.136 36.3438 183.831 36.0527C183.541 35.7617 183.156 35.5191 182.674 35.325C182.192 35.131 181.646 34.9612 181.035 34.8156C180.425 34.6701 179.807 34.5084 179.18 34.3305C178.537 34.1688 177.911 33.9747 177.301 33.7484C176.691 33.5058 176.144 33.1985 175.662 32.8266C175.18 32.4546 174.787 31.9937 174.482 31.444C174.192 30.8779 174.048 30.1988 174.048 29.4065C174.048 28.517 174.233 27.7489 174.602 27.1021C174.988 26.439 175.486 25.8893 176.096 25.4527C176.706 25.0161 177.397 24.6926 178.168 24.4824C178.939 24.2722 179.71 24.1671 180.481 24.1671C181.38 24.1671 182.24 24.2722 183.06 24.4824C183.895 24.6765 184.633 24.9998 185.276 25.4527C185.919 25.9054 186.425 26.4875 186.794 27.1991C187.18 27.8944 187.373 28.7434 187.373 29.7461H183.71C183.678 29.2285 183.565 28.8001 183.373 28.4605C183.196 28.1209 182.955 27.8541 182.65 27.66C182.344 27.466 181.992 27.3285 181.59 27.2476C181.204 27.1667 180.778 27.1264 180.313 27.1264C180.007 27.1264 179.703 27.1587 179.397 27.2234C179.091 27.288 178.811 27.4012 178.554 27.563C178.313 27.7246 178.112 27.9268 177.951 28.1694C177.79 28.412 177.71 28.7191 177.71 29.0911C177.71 29.4307 177.775 29.7056 177.903 29.9159C178.031 30.126 178.281 30.3201 178.65 30.498C179.036 30.6758 179.558 30.8537 180.216 31.0316C180.891 31.2095 181.766 31.4358 182.843 31.7108C183.164 31.7754 183.606 31.8967 184.168 32.0747C184.746 32.2363 185.317 32.5031 185.879 32.8751C186.44 33.247 186.922 33.7484 187.324 34.379C187.742 34.9935 187.951 35.7859 187.951 36.7562C187.951 37.5485 187.799 38.2843 187.493 38.9635C187.187 39.6427 186.73 40.2329 186.12 40.7342C185.525 41.2193 184.778 41.5993 183.879 41.8743C182.995 42.1491 181.968 42.2866 180.795 42.2866C179.847 42.2866 178.923 42.1653 178.024 41.9228C177.14 41.6963 176.353 41.3325 175.662 40.8312C174.988 40.3299 174.45 39.6912 174.048 38.915C173.646 38.1388 173.453 37.217 173.47 36.1497ZM195.672 39.4067C196.65 39.8183 197.695 40.0242 198.805 40.0242C199.914 40.0242 200.951 39.8183 201.915 39.4067C202.893 38.995 203.747 38.4217 204.478 37.6867C205.208 36.937 205.778 36.077 206.187 35.1067C206.61 34.1365 206.822 33.0927 206.822 31.9754C206.822 30.8581 206.61 29.8144 206.187 28.8442C205.778 27.8739 205.208 27.0212 204.478 26.2862C203.747 25.5511 202.893 24.9778 201.915 24.5662C200.951 24.1399 199.914 23.9267 198.805 23.9267C197.695 23.9267 196.65 24.1399 195.672 24.5662C194.708 24.9778 193.854 25.5511 193.109 26.2862C192.379 27.0212 191.809 27.8739 191.401 28.8442C190.992 29.8144 190.788 30.8581 190.788 31.9754C190.788 33.0927 190.992 34.1365 191.401 35.1067C191.809 36.077 192.379 36.937 193.109 37.6867C193.854 38.4217 194.708 38.995 195.672 39.4067ZM201.214 37.797C200.469 38.1204 199.667 38.2821 198.805 38.2821C197.928 38.2821 197.11 38.1204 196.351 37.797C195.607 37.4735 194.949 37.0252 194.38 36.4519C193.81 35.8785 193.365 35.2096 193.044 34.4452C192.723 33.6807 192.562 32.8575 192.562 31.9754C192.562 31.0934 192.723 30.2775 193.044 29.5278C193.365 28.7632 193.81 28.0944 194.38 27.5211C194.949 26.9478 195.607 26.4993 196.351 26.176C197.11 25.8525 197.928 25.6908 198.805 25.6908C199.667 25.6908 200.469 25.8525 201.214 26.176C201.973 26.4993 202.638 26.9478 203.207 27.5211C203.777 28.0944 204.223 28.7632 204.544 29.5278C204.865 30.2775 205.026 31.0934 205.026 31.9754C205.026 32.8575 204.865 33.6807 204.544 34.4452C204.223 35.2096 203.777 35.8785 203.207 36.4519C202.638 37.0252 201.973 37.4735 201.214 37.797ZM195.628 27.8519V36.1211H197.731V33.6954H198.98L199.834 35.1949C199.995 35.4595 200.207 35.6801 200.469 35.8565C200.746 36.0329 201.068 36.1211 201.433 36.1211H202.66V34.357H201.871C201.769 34.357 201.674 34.3055 201.587 34.2026L200.951 33.1442C201.36 32.8942 201.682 32.5634 201.915 32.1519C202.164 31.7255 202.288 31.2478 202.288 30.7185C202.288 30.1599 202.148 29.6673 201.871 29.2411C201.608 28.8147 201.25 28.4766 200.798 28.2267C200.36 27.9768 199.87 27.8519 199.33 27.8519H195.628ZM199.155 31.8211H197.731V29.7483H199.155C199.418 29.7483 199.651 29.8365 199.856 30.0129C200.075 30.1893 200.185 30.4245 200.185 30.7185C200.185 31.086 200.083 31.3653 199.878 31.5565C199.673 31.7329 199.432 31.8211 199.155 31.8211Z" fill="#0F0E12"/>
</svg>
</a><div class="header__burger"><button type="button"><svg width="21" height="14" viewBox="0 0 21 14" fill="none" xmlns="http://www.w3.org/2000/svg">
<line y1="1" x2="21" y2="1" stroke="black" stroke-width="2"/>
<line y1="7" x2="21" y2="7" stroke="black" stroke-width="2"/>
<line y1="13" x2="21" y2="13" stroke="black" stroke-width="2"/>
</svg>
</button></div><div class="header__close visually-hidden"><button type="button"><svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M1 1L17 17" stroke="#0F0E12" stroke-width="2" stroke-linecap="round"/>
<path d="M16.4287 1L1.00014 16.4286" stroke="#0F0E12" stroke-width="2" stroke-linecap="round"/>
</svg>
</button></div><div class="header__menu"><div class="header__links"><div class="header__link notes link"><a href="./software.html">Software</a></div><div class="header__link notes link"><a href="./kits.html">Kits</a></div><div class="header__link notes link"><a href="./services.html">Services</a></div><div class="header__link notes link"><a href="./customR&amp;D.html">Custom R&amp;D</a></div><div class="header__link notes link"><a href="./about.html">About</a></div><div class="header__link notes link inactive"><a href="./publications.html">Publications</a></div></div><div class="header__contact-link bordered-link"><a href="./contact.html">Contact</a></div><div class="header__mobile-bottom"><p>© 2016-2020 All rights reserved</p><svg width="80" height="60" viewBox="0 0 80 60" fill="none" xmlns="http://www.w3.org/2000/svg">
<rect width="80" height="60" fill="#0F0E12"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M34.4582 42.0833V40.6953H32.9931V42.0833H34.4582ZM11.9072 49.8712V46.0929H14.2719C14.6403 46.0929 14.9423 46.14 15.1779 46.2343C15.4135 46.3285 15.5998 46.4763 15.7369 46.6776C15.874 46.879 15.9682 47.1317 16.0197 47.4359C16.0711 47.74 16.0968 48.102 16.0968 48.5218C16.0968 48.7446 16.1075 48.9759 16.1289 49.2158C16.1503 49.4557 16.221 49.6742 16.3409 49.8712H18.063C17.8917 49.6827 17.7653 49.3679 17.6839 48.9266C17.6025 48.4854 17.5618 47.9949 17.5618 47.4552C17.5618 47.0953 17.5168 46.7954 17.4269 46.5556C17.3369 46.3157 17.2234 46.1208 17.0863 45.9708C16.9492 45.8209 16.8014 45.7116 16.6429 45.6431C16.4844 45.5746 16.3367 45.5317 16.1996 45.5146V45.4889C16.2767 45.4632 16.3988 45.4118 16.5658 45.3347C16.7329 45.2576 16.9021 45.1312 17.0735 44.9556C17.2448 44.7799 17.3948 44.5486 17.5233 44.2616C17.6518 43.9746 17.716 43.6126 17.716 43.1756C17.716 42.396 17.4633 41.7877 16.9578 41.3507C16.4523 40.9138 15.6941 40.6953 14.6831 40.6953H10.3008V49.8712H11.9072ZM14.5297 44.8728H11.8951V42.0069H14.504C14.7096 42.0069 14.9088 42.024 15.1016 42.0583C15.2943 42.0926 15.4657 42.1611 15.6156 42.2639C15.7655 42.3667 15.8834 42.5103 15.969 42.6945C16.0547 42.8787 16.0975 43.1164 16.0975 43.4077C16.0975 43.8961 15.9647 44.2623 15.6992 44.5065C15.4336 44.7507 15.0437 44.8728 14.5297 44.8728ZM23.1072 49.9031C22.7774 50.0016 22.4325 50.0509 22.0727 50.0509C21.5501 50.0509 21.0874 49.9652 20.6847 49.7939C20.2821 49.6225 19.9415 49.3826 19.6631 49.0742C19.3846 48.7658 19.1747 48.3973 19.0333 47.969C18.892 47.5406 18.8213 47.0694 18.8213 46.5553C18.8213 46.0841 18.8963 45.6364 19.0462 45.2123C19.1961 44.7882 19.4103 44.4156 19.6888 44.0943C19.9672 43.773 20.3035 43.5181 20.6976 43.3296C21.0917 43.1411 21.5372 43.0469 22.0341 43.0469C22.5568 43.0469 23.0258 43.1561 23.4414 43.3746C23.8569 43.5931 24.2017 43.8822 24.4759 44.2421C24.7501 44.6019 24.9493 45.0153 25.0735 45.4822C25.1977 45.9492 25.2299 46.4311 25.1699 46.928H20.2863C20.2863 47.185 20.3228 47.4335 20.3956 47.6734C20.4684 47.9133 20.5776 48.1232 20.7233 48.3031C20.8689 48.483 21.0531 48.6265 21.2759 48.7336C21.4987 48.8407 21.7643 48.8943 22.0727 48.8943C22.5011 48.8943 22.8459 48.8022 23.1072 48.618C23.3685 48.4338 23.5635 48.1575 23.692 47.789H25.0799C25.0028 48.1489 24.87 48.4702 24.6815 48.7529C24.493 49.0356 24.266 49.2734 24.0004 49.4662C23.7348 49.6589 23.4371 49.8046 23.1072 49.9031ZM23.7036 45.9641H20.2852C20.2937 45.7157 20.343 45.4844 20.4329 45.2702C20.5229 45.056 20.6429 44.8696 20.7928 44.7111C20.9427 44.5526 21.1226 44.4284 21.3325 44.3384C21.5425 44.2485 21.7759 44.2035 22.0329 44.2035C22.2814 44.2035 22.5063 44.2528 22.7076 44.3513C22.909 44.4498 23.0803 44.5783 23.2217 44.7368C23.3631 44.8953 23.4744 45.0817 23.5558 45.2959C23.6372 45.5101 23.6865 45.7328 23.7036 45.9641ZM29.9136 49.9352C29.5624 50.0123 29.2154 50.0509 28.8727 50.0509C28.4528 50.0509 28.0652 50.0038 27.7096 49.9095C27.3541 49.8153 27.0456 49.6718 26.7843 49.479C26.523 49.2862 26.3174 49.0463 26.1674 48.7593C26.0175 48.4723 25.9383 48.1317 25.9297 47.7376H27.3947C27.4376 48.166 27.6004 48.4659 27.8831 48.6372C28.1658 48.8086 28.5043 48.8943 28.8984 48.8943C29.0354 48.8943 29.1918 48.8836 29.3674 48.8621C29.5431 48.8407 29.708 48.8 29.8622 48.74C30.0164 48.6801 30.1428 48.5923 30.2413 48.4766C30.3399 48.3609 30.3848 48.2089 30.3763 48.0204C30.3677 47.8319 30.2992 47.6777 30.1707 47.5577C30.0421 47.4378 29.8772 47.3414 29.6759 47.2686C29.4745 47.1957 29.2453 47.1336 28.9883 47.0822C28.7313 47.0308 28.47 46.9751 28.2044 46.9152C27.9302 46.8552 27.6668 46.7824 27.414 46.6967C27.1613 46.611 26.9342 46.4953 26.7329 46.3497C26.5316 46.204 26.3709 46.0177 26.251 45.7907C26.131 45.5636 26.0711 45.283 26.0711 44.9489C26.0711 44.589 26.1589 44.287 26.3345 44.0429C26.5101 43.7987 26.7329 43.6016 27.0028 43.4517C27.2727 43.3018 27.5725 43.1968 27.9024 43.1368C28.2322 43.0769 28.5471 43.0469 28.847 43.0469C29.1897 43.0469 29.5174 43.0833 29.8301 43.1561C30.1428 43.2289 30.4255 43.3467 30.6783 43.5095C30.931 43.6723 31.1409 43.8844 31.308 44.1457C31.4751 44.407 31.58 44.7218 31.6229 45.0903H30.0935C30.025 44.739 29.8644 44.5034 29.6116 44.3834C29.3589 44.2635 29.0697 44.2035 28.7441 44.2035C28.6413 44.2035 28.5192 44.2121 28.3779 44.2292C28.2365 44.2463 28.1037 44.2785 27.9795 44.3256C27.8553 44.3727 27.7503 44.4413 27.6646 44.5312C27.579 44.6212 27.5361 44.739 27.5361 44.8846C27.5361 45.0645 27.5982 45.2102 27.7225 45.3216C27.8467 45.433 28.0095 45.5251 28.2108 45.5979C28.4122 45.6707 28.6413 45.7328 28.8984 45.7842C29.1554 45.8356 29.421 45.8913 29.6952 45.9513C29.9607 46.0113 30.2221 46.0841 30.4791 46.1698C30.7361 46.2554 30.9653 46.3711 31.1666 46.5168C31.368 46.6624 31.5308 46.8466 31.655 47.0694C31.7792 47.2921 31.8413 47.5663 31.8413 47.8919C31.8413 48.286 31.7514 48.6201 31.5715 48.8943C31.3915 49.1684 31.1581 49.3912 30.8711 49.5625C30.584 49.7339 30.2649 49.8581 29.9136 49.9352ZM34.4582 43.2266V49.8707H32.9932V43.2266H34.4582ZM38.8319 50.0511C39.2518 50.0511 39.6502 49.9676 40.0271 49.8005C40.4041 49.6335 40.6911 49.3572 40.8882 48.9716H40.9139V49.8712H42.3018V40.6953H40.8368V44.0881H40.8111C40.6997 43.9082 40.5605 43.7518 40.3934 43.619C40.2263 43.4862 40.0443 43.377 39.8472 43.2913C39.6502 43.2056 39.4445 43.1435 39.2303 43.1049C39.0161 43.0664 38.8062 43.0471 38.6006 43.0471C38.2408 43.0471 37.8874 43.1135 37.5404 43.2463C37.1934 43.3791 36.885 43.5869 36.6151 43.8696C36.3452 44.1523 36.1289 44.5122 35.9661 44.9491C35.8033 45.3861 35.7219 45.9044 35.7219 46.5041C35.7219 47.0011 35.7862 47.4659 35.9147 47.8985C36.0432 48.3312 36.236 48.706 36.493 49.023C36.75 49.34 37.0734 49.5906 37.4633 49.7748C37.8531 49.959 38.3093 50.0511 38.8319 50.0511ZM38.2027 48.7015C38.434 48.83 38.7082 48.8943 39.0252 48.8943C39.3508 48.8943 39.6314 48.8257 39.867 48.6886C40.1026 48.5516 40.2954 48.3716 40.4453 48.1489C40.5952 47.9261 40.7066 47.6755 40.7794 47.3971C40.8522 47.1186 40.8887 46.8338 40.8887 46.5425C40.8887 45.8056 40.7237 45.2316 40.3939 44.8204C40.064 44.4091 39.6164 44.2035 39.0509 44.2035C38.7082 44.2035 38.419 44.2742 38.1834 44.4155C37.9478 44.5569 37.7551 44.7411 37.6051 44.9682C37.4552 45.1952 37.3481 45.4522 37.2838 45.7392C37.2196 46.0263 37.1875 46.3197 37.1875 46.6196C37.1875 46.9023 37.2239 47.1807 37.2967 47.4549C37.3695 47.7291 37.4809 47.9732 37.6308 48.1874C37.7808 48.4016 37.9714 48.573 38.2027 48.7015ZM46.8334 50.0509C47.1933 50.0509 47.5381 50.0016 47.868 49.9031C48.1978 49.8046 48.4955 49.6589 48.7611 49.4662C49.0267 49.2734 49.2538 49.0356 49.4423 48.7529C49.6308 48.4702 49.7636 48.1489 49.8407 47.789H48.4527C48.3242 48.1575 48.1293 48.4338 47.868 48.618C47.6067 48.8022 47.2618 48.8943 46.8334 48.8943C46.525 48.8943 46.2594 48.8407 46.0366 48.7336C45.8139 48.6265 45.6297 48.483 45.484 48.3031C45.3384 48.1232 45.2292 47.9133 45.1563 47.6734C45.0835 47.4335 45.0471 47.185 45.0471 46.928H49.9306C49.9906 46.4311 49.9585 45.9492 49.8342 45.4822C49.71 45.0153 49.5108 44.6019 49.2366 44.2421C48.9625 43.8822 48.6176 43.5931 48.2021 43.3746C47.7866 43.1561 47.3175 43.0469 46.7949 43.0469C46.298 43.0469 45.8524 43.1411 45.4583 43.3296C45.0642 43.5181 44.7279 43.773 44.4495 44.0943C44.1711 44.4156 43.9569 44.7882 43.8069 45.2123C43.657 45.6364 43.582 46.0841 43.582 46.5553C43.582 47.0694 43.6527 47.5406 43.7941 47.969C43.9354 48.3973 44.1454 48.7658 44.4238 49.0742C44.7022 49.3826 45.0428 49.6225 45.4455 49.7939C45.8482 49.9652 46.3108 50.0509 46.8334 50.0509ZM45.0462 45.9641H48.4646C48.4475 45.7328 48.3982 45.5101 48.3168 45.2959C48.2354 45.0817 48.1241 44.8953 47.9827 44.7368C47.8413 44.5783 47.67 44.4498 47.4686 44.3513C47.2673 44.2528 47.0424 44.2035 46.7939 44.2035C46.5369 44.2035 46.3035 44.2485 46.0935 44.3384C45.8836 44.4284 45.7037 44.5526 45.5538 44.7111C45.4039 44.8696 45.2839 45.056 45.1939 45.2702C45.104 45.4844 45.0547 45.7157 45.0462 45.9641ZM52.5002 49.871V45.9513C52.5002 45.7114 52.5388 45.4844 52.6159 45.2702C52.693 45.056 52.8001 44.8696 52.9372 44.7111C53.0742 44.5526 53.2413 44.4284 53.4384 44.3384C53.6354 44.2485 53.8582 44.2035 54.1066 44.2035C54.5436 44.2035 54.8649 44.3213 55.0705 44.5569C55.2761 44.7925 55.3875 45.1716 55.4046 45.6943V49.871H56.8697V45.3087C56.8697 44.5548 56.6598 43.9893 56.24 43.6123C55.8202 43.2354 55.2461 43.0469 54.5179 43.0469C54.0809 43.0469 53.6825 43.1518 53.3227 43.3617C52.9629 43.5716 52.6716 43.8608 52.4488 44.2292L52.4231 44.2035V43.2268H51.0352V49.871H52.5002ZM61.172 49.9097C61.0178 49.9268 60.8593 49.9354 60.6965 49.9354C60.3109 49.9354 60.0004 49.8989 59.7647 49.8261C59.5291 49.7533 59.3449 49.6462 59.2121 49.5048C59.0793 49.3635 58.9894 49.1857 58.9423 48.9715C58.8951 48.7573 58.8673 48.5131 58.8587 48.239V44.3193H57.7535V43.2269H58.8587V41.235H60.3238V43.2269H61.6475V44.3193H60.3238V47.8663C60.3238 48.0205 60.3302 48.1533 60.3431 48.2647C60.3559 48.3761 60.3859 48.4703 60.433 48.5474C60.4801 48.6245 60.5508 48.6823 60.6451 48.7209C60.7393 48.7595 60.8678 48.7787 61.0306 48.7787C61.1334 48.7787 61.2362 48.7766 61.339 48.7723C61.4419 48.768 61.5447 48.753 61.6475 48.7273V49.8583C61.4847 49.8754 61.3262 49.8925 61.172 49.9097Z" fill="#E6E7EA"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M24.1732 35.1469C22.8656 35.5474 21.3518 35.7477 19.6319 35.7477C18.2418 35.7477 16.8929 35.5768 15.5853 35.2352C14.2776 34.8936 13.1232 34.3576 12.1218 33.6272C11.1205 32.8968 10.3253 31.9661 9.73627 30.8352C9.14725 29.7043 8.86452 28.3613 8.88808 26.8063H14.26C14.26 27.6545 14.4131 28.3731 14.7194 28.9621C15.0257 29.5511 15.4321 30.0282 15.9387 30.3934C16.4452 30.7586 17.0402 31.0296 17.7234 31.2063C18.4067 31.383 19.1135 31.4714 19.8439 31.4714C20.3387 31.4714 20.8688 31.4301 21.4343 31.3477C21.9997 31.2652 22.5298 31.1062 23.0246 30.8706C23.5194 30.6349 23.9317 30.311 24.2616 29.8987C24.5914 29.4863 24.7564 28.9621 24.7564 28.326C24.7564 27.6427 24.5384 27.089 24.1025 26.6649C23.6667 26.2408 23.0953 25.8874 22.3885 25.6047C21.6817 25.322 20.8806 25.0746 19.9853 24.8625C19.09 24.6505 18.1829 24.4149 17.264 24.1557C16.3215 23.9201 15.4027 23.6315 14.5074 23.2898C13.612 22.9482 12.811 22.5064 12.1041 21.9645C11.3973 21.4226 10.826 20.7452 10.3901 19.9324C9.95421 19.1195 9.73627 18.1359 9.73627 16.9814C9.73627 15.6855 10.0131 14.5605 10.5668 13.6063C11.1205 12.6521 11.845 11.8569 12.7403 11.2207C13.6356 10.5846 14.6487 10.1134 15.7796 9.80709C16.9106 9.5008 18.0415 9.34766 19.1724 9.34766C20.4918 9.34766 21.7582 9.49491 22.9716 9.78942C24.185 10.0839 25.2629 10.561 26.2053 11.2207C27.0062 11.7814 27.6668 12.4739 28.1869 13.2982V9.94832H33.2054V23.4841L39.5315 16.9106H45.4688L38.5773 23.6254L46.2463 35.1821H40.1676L35.1491 27.0182L33.2054 28.8913V35.1821H28.1869V32.7948C27.9637 33.0311 27.7214 33.2555 27.46 33.4681C26.5764 34.1867 25.4809 34.7463 24.1732 35.1469ZM28.1869 22.9297C27.8461 22.5904 27.4801 22.2923 27.0889 22.0352C26.2643 21.4933 25.4337 21.0987 24.5973 20.8513C23.7609 20.6039 23.1071 20.4331 22.6359 20.3388C21.0573 19.9383 19.7791 19.6084 18.8013 19.3493C17.8236 19.0901 17.0578 18.8309 16.5041 18.5718C15.9505 18.3126 15.5794 18.0299 15.3909 17.7236C15.2024 17.4173 15.1082 17.0167 15.1082 16.522C15.1082 15.9801 15.226 15.5324 15.4616 15.179C15.6972 14.8256 15.9976 14.5311 16.3628 14.2954C16.728 14.0598 17.1344 13.8949 17.5821 13.8007C18.0297 13.7064 18.4774 13.6593 18.925 13.6593C19.6083 13.6593 20.2385 13.7182 20.8158 13.836C21.393 13.9538 21.9055 14.1541 22.3531 14.4368C22.8008 14.7195 23.1601 15.1083 23.4311 15.6031C23.702 16.0979 23.861 16.7222 23.9082 17.4762H28.1869V22.9297Z" fill="#E6E7EA"/>
</svg>
</div></div></div></header><main class="content"><div class="publications container"><div class="subtitle notes"><a href="./home.html">↲ INDEX</a></div><div class="title title--no-text"><div class="title__left h1">Publications</div></div><div class="publications__list"><div class="publications__list-item"><div class="h2">2020</div><div class="publications__list-item-inner"><p class="h3">B cells, plasma cells and antibody repertoires in the tumour microenvironment. Nat Rev Immunol.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Sharonov GV, Serebrovskaya EO, Yuzhakova DV, Britanova OV, Chudakov DM.</p><div class="notes publications__link"><a href="https://www.nature.com/articles/s41577-019-0257-x" class="notes--link" target="_blank">doi: 10.1038/s41577-019-0257-x.↗</a></div><div class="notes">2020 Jan 27</div></div><div class="publications__article">Recent data show that B cells and plasma cells located in tumours or in tumour-draining lymph nodes can have important roles in shaping antitumour immune responses. In tumour-associated tertiary lymphoid structures, T cells and B cells interact and undergo cooperative selection, specialization and clonal expansion. Importantly, B cells can present cognate tumour-derived antigens to T cells, with the functional consequences of such interactions being shaped by the B cell phenotype. Furthermore, the isotype and specificity of the antibodies produced by plasma cells can drive distinct immune responses. Here we summarize our current knowledge of the roles of B cells and antibodies in the tumour microenvironment. Moreover, we discuss the potential of using immunoglobulin repertoires as a source of tumour-specific receptors for immunotherapy or as biomarkers to predict the efficacy of immunotherapeutic interventions.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">An overview of immunoinformatics approaches and databases linking T cell receptor repertoires to their antigen specificity. immunogenetics.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Zvyagin IV, Tsvetkov VO, Chudakov DM, Shugay M.</p><div class="notes publications__link">72(1-2):77-84. <a href="https://link.springer.com/article/10.1007%2Fs00251-019-01139-4" class="notes--link" target="_blank">doi: 10.1007/s00251-019-01139-4.↗</a></div><div class="notes">2020 Feb</div></div><div class="publications__article">Recent advances in molecular and bioinformatic methods have greatly improved our ability to study the formation of an adaptive immune response towards foreign pathogens, self-antigens, and cancer neoantigens. T cell receptors (TCR) are the key players in this process that recognize peptides presented by major histocompatibility complex (MHC). Owing to the huge diversity of both TCR sequence variants and peptides they recognize, accumulation and complex analysis of large amounts of TCR-antigen specificity data is required for understanding the structure and features of adaptive immune responses towards pathogens, vaccines, cancer, as well as autoimmune responses. In the present review, we summarize recent efforts on gathering and interpreting TCR-antigen specificity data and outline the critical role of tighter integration with other immunoinformatics data sources that include epitope MHC restriction, TCR repertoire structure models, and TCR/peptide/MHC structural data. We suggest that such integration can lead to the ability to accurately annotate individual TCR repertoires, efficiently estimate epitope and neoantigen immunogenicity, and ultimately, in silico identify TCRs specific to yet unstudied antigens and predict self-peptides related to autoimmunity.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="h2">2019</div><div class="publications__list-item-inner"><p class="h3">Intratumoral immunoglobulin isotypes predict survival in lung adenocarcinoma subtypes. J Immunother Cancer.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Isaeva OI, Sharonov GV, Serebrovskaya EO, Turchaninova MA, Zaretsky AR, Shugay M, Chudakov DM.</p><div class="notes publications__link">7(1):279. <a href="https://jitc.biomedcentral.com/articles/10.1186/s40425-019-0747-1" class="notes--link" target="_blank">doi: 10.1186/s40425-019-0747-1.↗</a></div><div class="notes">2019 Oct 29</div></div><div class="publications__article">Background<br>
The role of tumor-infiltrating B-cells (TIBs) and intratumorally-produced antibodies in cancer-immunity interactions essentially remains terra incognita. In particular, it remains unexplored how driver mutations could be associated with distinct TIBs signatures and their role in tumor microenvironment.<br><br>Methods<br>
Here we analyzed associations of immunoglobulin isotypes and clonality with survival in TCGA RNA-Seq data for lung adenocarcinoma (LUAD), stratifying patients into 12 driver mutation and phenotypic tumor subgroups.<br><br>Results<br>
We revealed several unexpected associations between TIBs behavior and prognosis. Abundance and high proportion of IgG1 isotype, and low proportion of IgA among all intratumorally produced immunoglobulins were specifically associated with improved overall survival for KRASmut but not KRASwt LUAD, revealing the first link between a driver mutation and B-cell response. We found specific IgG1 signature associated with long survival, which suggests that particular specificities of IgG1+ TIBs could be beneficial in KRASmut LUAD. In contrast to our previous observations for melanoma, highly clonal IgG1 production by plasma cells had no meaningful effect on prognosis, suggesting that IgG1+ TIBs may exert a beneficial effect in KRASmut cases in an alternative way, such as efficient presentation of cognate antigens or direct B cell attack on tumor cells. Notably, a high proportion of the IgG1 isotype is positively correlated with the non-silent mutation burden both in the general LUAD cohort and in most patient subgroups, supporting a role for IgG1+ TIBs in antigen presentation. Complementing the recent finding that the presence of stromal IgG4-producing cells is associated with a favorable prognosis for patients with stage I squamous cell carcinoma, we show that the abundance of IgG4-producing TIBs likewise has a strong positive effect on overall survival in STK11mut and proximal proliferative subgroups of LUAD patients. We hypothesize that the positive role of IgG4 antibodies in some of the lung cancer subtypes could be associated with reported inability of IgG4 isotype to form immune complexes, thus preventing immunosuppression via activation of the myeloid-derived suppressor cell (MDSC) phenotype.<br><br>Conclusions<br>
We discover prominent and distinct associations between TIBs antibody isotypes and survival in lung adenocarcinoma carrying specific driver mutations. These findings indicate that particular types of tumor-immunity relations could be beneficial in particular driver mutation context, which should be taken into account in developing strategies of cancer immunotherapy and combination therapies. Specificity of protective B cell populations in specific cancer subgroups could become a clue to efficient targeted immunotherapies for appropriate cohorts of patients.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">VDJdb in 2019: database extension, new analysis infrastructure and a T-cell receptor motif compendium. Nucleic Acids Res.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Bagaev DV, Vroomans RMA, Samir J, Stervbo U, Rius C, Dolton G, Greenshields-Watson A, Attaf M, Egorov ES, Zvyagin IV, Babel N, Cole DK, Godkin AJ, Sewell AK, Kesmir C, Chudakov DM, Luciani F, Shugay M</p><div class="notes publications__link">pii: gkz874. <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz874/5582255" class="notes--link" target="_blank">doi: 10.1093/nar/gkz874.↗</a></div><div class="notes">2019 Oct 7</div></div><div class="publications__article">Here, we report an update of the VDJdb database with a substantial increase in the number of T-cell receptor (TCR) sequences and their cognate antigens. The update further provides a new database infrastructure featuring two additional analysis modes that facilitate database querying and real-world data analysis. The increased yield of TCR specificity identification methods and the overall increase in the number of studies in the field has allowed us to expand the database more than 5-fold. Furthermore, several new analysis methods are included. For example, batch annotation of TCR repertoire sequencing samples allows for annotating large datasets on-line. Using recently developed bioinformatic methods for TCR motif mining, we have built a reduced set of high-quality TCR motifs that can be used for both training TCR specificity predictors and matching against TCRs of interest. These additions enhance the versatility of the VDJdb in the task of exploring T-cell antigen specificities. The database is available at <a href="https://vdjdb.cdr3.net" class="notes--link" target="_blank">https://vdjdb.cdr3.net</a>.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Detecting T cell receptors involved in immune responses from single repertoire snapshots. PLoS Biol.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Pogorelyy MV, Minervina AA, Shugay M, Chudakov DM, Lebedev YB, Mora T, Walczak AM.</p><div class="notes publications__link">17(6):e3000314. <a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3000314" class="notes--link" target="_blank">doi: 10.1371/journal.pbio.3000314.↗</a></div><div class="notes">2019 Jun 13</div></div><div class="publications__article">Hypervariable T cell receptors (TCRs) play a key role in adaptive immunity, recognizing a vast diversity of pathogen-derived antigens. Our ability to extract clinically relevant information from large high-throughput sequencing of TCR repertoires (RepSeq) data is limited, because little is known about TCR–disease associations. We present Antigen-specific Lymphocyte Identification by Clustering of Expanded sequences (ALICE), a statistical approach that identifies TCR sequences actively involved in current immune responses from a single RepSeq sample and apply it to repertoires of patients with a variety of disorders — patients with autoimmune disease (ankylosing spondylitis [AS]), under cancer immunotherapy, or subject to an acute infection (live yellow fever [YF] vaccine). We validate the method with independent assays. ALICE requires no longitudinal data collection nor large cohorts, and it is directly applicable to most RepSeq datasets. Its results facilitate the identification of TCR variants associated with diseases and conditions, which can be used for diagnostics and rational vaccine design.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Memory CD4+ T cells are generated in the human fetal intestine. Nature Immunology.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Li N, van Unen V, Abdelaal T, Guo N, Kasatskaya SA, Ladell K, McLaren JE, Egorov ES, Izraelson M, Chuva de Sousa Lopes SM, Höllt T, Britanova OV, Eggermont J, de Miranda NFCC, Chudakov DM, Price DA, Lelieveldt BPF, Koning F.</p><div class="notes publications__link">20(3):301-312. <a href="https://www.nature.com/articles/s41590-018-0294-9" class="notes--link" target="_blank">doi: 10.1038/s41590-018-0294-9.↗</a></div><div class="notes">2019 Mar</div></div><div class="publications__article">The fetus is thought to be protected from exposure to foreign antigens, yet CD45RO+ T cells reside in the fetal intestine. Here we combined functional assays with mass cytometry, single-cell RNA sequencing and high-throughput T cell antigen receptor (TCR) sequencing to characterize the CD4+ T cell compartment in the human fetal intestine. We identified 22 CD4+ T cell clusters, including naive-like, regulatory-like and memory-like subpopulations, which were confirmed and further characterized at the transcriptional level. Memory-like CD4+ T cells had high expression of Ki-67, indicative of cell division, and CD5, a surrogate marker of TCR avidity, and produced the cytokines IFN-γ and IL-2. Pathway analysis revealed a differentiation trajectory associated with cellular activation and proinflammatory effector functions, and TCR repertoire analysis indicated clonal expansions, distinct repertoire characteristics and interconnections between subpopulations of memory-like CD4+ T cells. Imaging mass cytometry indicated that memory-like CD4+ T cells colocalized with antigen-presenting cells. Collectively, these results provide evidence for the generation of memory-like CD4+ T cells in the human fetal intestine that is consistent with exposure to foreign antigens.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="h2">2018</div><div class="publications__list-item-inner"><p class="h3">Exploring the pre-immune landscape of antigen-specific T cells. Genome Medicine.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Pogorelyy MV, Fedorova AD, McLaren JE, Ladell K, Bagaev DV, Eliseev AV, Mikelov AI, Koneva AE, Zvyagin IV, Price DA, Chudakov DM, Shugay M.</p><div class="notes publications__link">10(1):68. <a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-018-0577-7" class="notes--link" target="_blank">doi: 10.1186/s13073-018-0577-7.↗</a></div><div class="notes">2018 Aug 25</div></div><div class="publications__article">Background<br>
Adaptive immune responses to newly encountered pathogens depend on the mobilization of antigen-specific clonotypes from a vastly diverse pool of naive T cells. Using recent advances in immune repertoire sequencing technologies, models of the immune receptor rearrangement process, and a database of annotated T cell receptor (TCR) sequences with known specificities, we explored the baseline frequencies of T cells specific for defined human leukocyte antigen (HLA) class I-restricted epitopes in healthy individuals.<br><br>Methods<br>
We used a database of TCR sequences with known antigen specificities and a probabilistic TCR rearrangement model to estimate the baseline frequencies of TCRs specific to distinct antigens epitopespecificT-cells. We verified our estimates using a publicly available collection of TCR repertoires from healthy individuals. We also interrogated a database of immunogenic and non-immunogenic peptides is used to link baseline T-cell frequencies with epitope immunogenicity.<br><br>Results<br>
 Our findings revealed a high degree of variability in the prevalence of T cells specific for different antigens that could be explained by the physicochemical properties of the corresponding HLA class I-bound peptides. The occurrence of certain rearrangements was influenced by ancestry and HLA class I restriction, and umbilical cord blood samples contained higher frequencies of common pathogen-specific TCRs. We also identified a quantitative link between specific T cell frequencies and the immunogenicity of cognate epitopes presented by defined HLA class I molecules.<br><br>Conclusions<br>
Our results suggest that the population frequencies of specific T cells are strikingly non-uniform across epitopes that are known to elicit immune responses. This inference leads to a new definition of epitope immunogenicity based on specific TCR frequencies, which can be estimated with a high degree of accuracy in silico, thereby providing a novel framework to integrate computational and experimental genomics with basic and translational research efforts in the field of T cell immunology.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">The changing landscape of naive T cell receptor repertoire with human aging. Frontiers in Immunology.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Egorov ES, Kasatskaya SA, Zubov VN, Izraelson M, Nakonechnaya TO, Staroverov DB, Angius A, Cucca F, Mamedov IZ, Rosati E, Franke A, Shugay M, Pogorelyy MV, Chudakov DM, Britanova OV.</p><div class="notes publications__link">9:1618. <a href="https://www.frontiersin.org/articles/10.3389/fimmu.2018.01618/full" class="notes--link" target="_blank">doi: 10.3389/fimmu.2018.01618.↗</a></div><div class="notes">2018 Jul 24</div></div><div class="publications__article">Human aging is associated with a profound loss of thymus productivity, yet naïve T lymphocytes still maintain their numbers by division in the periphery for many years. The extent of such proliferation may depend on the cytokine environment, including IL-7 and T-cell receptor (TCR) "tonic" signaling mediated by self pMHCs recognition. Additionally, intrinsic properties of distinct subpopulations of naïve T cells could influence the overall dynamics of aging-related changes within the naïve T cell compartment. Here, we investigated the differences in the architecture of TCR beta repertoires for naïve CD4, naïve CD8, naïve CD4+CD25-CD31+ (enriched with recent thymic emigrants, RTE), and mature naïve CD4+CD25-CD31- peripheral blood subsets between young and middle-age/old healthy individuals. In addition to observing the accumulation of clonal expansions (as was shown previously), we reveal several notable changes in the characteristics of T cell repertoire. We observed significant decrease of CDR3 length, NDN insert, and number of non-template added N nucleotides within TCR beta CDR3 with aging, together with a prominent change of physicochemical properties of the central part of CDR3 loop. These changes were similar across CD4, CD8, RTE-enriched, and mature CD4 subsets of naïve T cells, with minimal or no difference observed between the latter two subsets for individuals of the same age group. We also observed an increase in "publicity" (fraction of shared clonotypes) of CD4, but not CD8 naïve T cell repertoires. We propose several explanations for these phenomena built upon previous studies of naïve T-cell homeostasis, and call for further studies of the mechanisms causing the observed changes and of consequences of these changes in respect of the possible holes formed in the landscape of naïve T cell TCR repertoire.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Human liver infiltrating γδ T cells are composed of clonally expanded circulating and tissue-resident populations. Journal of Hepatology.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Hunter S, Willcox CR, Davey MS, Kasatskaya SA, Jeffery HC, Chudakov DM, Oo YH, Willcox BE.</p><div class="notes publications__link">69(3):654-665. <a href="https://www.journal-of-hepatology.eu/article/S0168-8278(18)32053-1/fulltext" class="notes--link" target="_blank">doi: 10.1016/j.jhep.2018.05.007.↗</a></div><div class="notes">2018 Sep</div></div><div class="publications__article">BACKGROUND &amp; AIMS:<br>
γδ T cells comprise a substantial proportion of tissue-associated lymphocytes. However, our current understanding of human γδ T cells is primarily based on peripheral blood subsets, while the immunobiology of tissue-associated subsets remains largely unclear. Therefore, we aimed to elucidate the T cell receptor (TCR) diversity, immunophenotype and function of γδ T cells in the human liver.<br><br>METHODS:<br>
We characterised the TCR repertoire, immunophenotype and function of human liver infiltrating γδ T cells, by TCR sequencing analysis, flow cytometry, in situ hybridisation and immunohistochemistry. We focussed on the predominant tissue-associated Vδ2- γδ subset, which is implicated in liver immunopathology.<br><br>RESULTS:<br>
Intrahepatic Vδ2- γδ T cells were highly clonally focussed, with single expanded clonotypes featuring complex, private TCR rearrangements frequently dominating the compartment. Such T cells were predominantly CD27lo/- effector lymphocytes, whereas naïve CD27hi, TCR-diverse populations present in matched blood were generally absent in the liver. Furthermore, while a CD45RAhi Vδ2- γδ effector subset present in both liver and peripheral blood contained overlapping TCR clonotypes, the liver Vδ2- γδ T cell pool also included a phenotypically distinct CD45RAlo effector compartment that was enriched for expression of the tissue tropism marker CD69, the hepatic homing chemokine receptors CXCR3 and CXCR6, and liver-restricted TCR clonotypes, suggestive of intrahepatic tissue residency. Liver infiltrating Vδ2- γδ cells were capable of polyfunctional cytokine secretion, and unlike peripheral blood subsets, were responsive to both TCR and innate stimuli.<br><br>CONCLUSION:<br>
These findings suggest that the ability of Vδ2- γδ T cells to undergo clonotypic expansion and differentiation is crucial in permitting access to solid tissues, such as the liver, which results in functionally distinct peripheral and liver-resident memory γδ T cell subsets. They also highlight the inherent functional plasticity within the Vδ2- γδ T cell compartment and provide information that could be used for the design of cellular therapies that suppress liver inflammation or combat liver cancer.<br><br>LAY SUMMARY:<br>
γδ T cells are frequently enriched in many solid tissues, however the immunobiology of such tissue-associated subsets in humans has remained unclear. We show that intrahepatic γδ T cells are enriched for clonally expanded effector T cells, whereas naïve γδ T cells are largely excluded. Moreover, whereas a distinct proportion of circulating T cell clonotypes was present in both the liver tissue and peripheral blood, a functionally and clonotypically distinct population of liver-resident γδ T cells was also evident. Our findings suggest that factors triggering γδ T cell clonal selection and differentiation, such as infection, can drive enrichment of γδ T cells into liver tissue, allowing the development of functionally distinct tissue-restricted memory populations specialised in local hepatic immunosurveillance.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">The human Vδ2+ T-cell compartment comprises distinct innate-like Vγ9+ and adaptive Vγ9- subsets. Nature Communications.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Davey MS, Willcox CR, Hunter S, Kasatskaya SA, Remmerswaal EBM, Salim M, Mohammed F, Bemelman FJ, Chudakov DM, Oo YH, Willcox BE.</p><div class="notes publications__link">9(1):1760. <a href="https://www.nature.com/articles/s41467-018-04076-0 class="notes--link" target="_blank">doi: 10.1038/s41467-018-04076-0.↗</a></div><div class="notes">2018 May 2</div></div><div class="publications__article">Vδ2+ T cells form the predominant human γδ T-cell population in peripheral blood and mediate T-cell receptor (TCR)-dependent anti-microbial and anti-tumour immunity. Here we show that the Vδ2+ compartment comprises both innate-like and adaptive subsets. Vγ9+ Vδ2+ T cells display semi-invariant TCR repertoires, featuring public Vγ9 TCR sequences equivalent in cord and adult blood. By contrast, we also identify a separate, Vγ9- Vδ2+ T-cell subset that typically has a CD27hiCCR7+CD28+IL-7Rα+ naive-like phenotype and a diverse TCR repertoire, however in response to viral infection, undergoes clonal expansion and differentiation to a CD27loCD45RA+CX3CR1+granzymeA/B+ effector phenotype. Consistent with a function in solid tissue immunosurveillance, we detect human intrahepatic Vγ9- Vδ2+ T cells featuring dominant clonal expansions and an effector phenotype. These findings redefine human γδ T-cell subsets by delineating the Vδ2+ T-cell compartment into innate-like (Vγ9+) and adaptive (Vγ9-) subsets, which have distinct functions in microbial immunosurveillance.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Method for identification of condition-associated public antigen receptor sequences. Elife.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Pogorelyy MV, Minervina AA, Chudakov DM, Mamedov IZ, Lebedev YB, Mora T, Walczak AM.</p><div class="notes publications__link">7. pii: e33050. <a href="https://elifesciences.org/articles/33050" class="notes--link" target="_blank">doi: 10.7554/eLife.33050.↗</a></div><div class="notes">2018 Mar 13</div></div><div class="publications__article">Diverse repertoires of hypervariable immunoglobulin receptors (TCR and BCR) recognize antigens in the adaptive immune system. The development of immunoglobulin receptor repertoire sequencing methods makes it possible to perform repertoire-wide disease association studies of antigen receptor sequences. We developed a statistical framework for associating receptors to disease from only a small cohort of patients, with no need for a control cohort. Our method successfully identifies previously validated Cytomegalovirus and type one diabetes responsive TCR sequences.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">CD8+ T cells with characteristic T cell receptor beta motif are detected in blood and expanded in synovial fluid of ankylosing spondylitis patients. Rheumatology (Oxford).<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Komech EA, Pogorelyy MV, Egorov ES, Britanova OV, Rebrikov DV, Bochkova AG, Shmidt EI, Shostak NA, Shugay M, Lukyanov S, Mamedov IZ, Lebedev YB, Chudakov DM, Zvyagin IV.</p><div class="notes publications__link"><a href="https://academic.oup.com/rheumatology/advance-article-abstract/doi/10.1093/rheumatology/kex517/4903020?redirectedFrom=fulltext" class="notes--link" target="_blank">doi: 10.1093/rheumatology/kex517.↗</a></div><div class="notes">2018 Feb 22</div></div><div class="publications__article">The risk of AS is associated with genomic variants related to antigen presentation and specific cytokine signalling pathways, suggesting the involvement of cellular immunity in disease initiation/progression. The aim of the present study was to explore the repertoire of TCR sequences in healthy donors and AS patients to uncover AS-linked TCR variants.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Quantitative profiling reveals minor changes of T cell receptor repertoire in response to subunit inactivated influenza vaccine. Vaccine.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Sycheva AL, Pogorelyy MV, Komech EA, Minervina AA, Zvyagin IV, Staroverov DB, Chudakov DM, Lebedev YB, Mamedov IZ.</p><div class="notes publications__link">36(12):1599-1605 <a href="https://www.sciencedirect.com/science/article/pii/S0264410X18302019?via%3Dihub" class="notes--link" target="_blank">doi: 10.1016/j.vaccine.2018.02.027.↗</a></div><div class="notes">2018 Mar 14</div></div><div class="publications__article">Vaccination against influenza is widely used to protect against seasonal flu epidemic although its effectiveness is debated. Here we performed deep quantitative T cell receptor repertoire profiling in peripheral blood of a healthy volunteer in response to trivalent subunit influenza vaccine. We did not observe significant rebuilding of peripheral blood T cell receptors composition in response to vaccination. However, we found several clonotypes in memory T cell fraction that were undetectable before the vaccination and had a maximum concentration at day 45 after vaccine administration. These cells were found in lower concentration in the course of repertoire monitoring for two years period. Our observation suggests a potential for recruitment of only a limited number of new T cells after each seasonal influenza vaccination.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="h2">2017</div><div class="publications__list-item-inner"><p class="h3">Comparative Analysis of Murine T Cell Receptor Repertoires. Immunology.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Izraelson M, Nakonechnaya TO, Moltedo B, Egorov ES, Kasatskaya SA, Putintseva EV, Mamedov IZ, Staroverov DB, Shemiakina II, Zakharova MY, Davydov AN, Bolotin DA, Shugay M, Chudakov DM, Rudensky AY, Britanova OV.</p><div class="notes publications__link"><a href="http://onlinelibrary.wiley.com/doi/10.1111/imm.12857/abstract;jsessionid=F6EAB84967C08617FB22C662485E9B2F.f04t04" class="notes--link" target="_blank">doi: 10.1111/imm.12857.↗</a></div><div class="notes">2017 Oct 28</div></div><div class="publications__article">On the way of understanding the rules and laws of adaptive immunity, high-throughput profiling of TCR repertoires becomes a powerful tool. The structure of TCR repertoires is insightful and instructive even before antigen specificity of each particular receptor becomes available. It embodies information about the thymic and peripheral selection of T cells, about the readiness of an adaptive immunity to withstand new challenges, about the character, the magnitude, and the memory of immune response, and about the etiological and functional proximity of T cell subsets. Here, we describe our current analytical approaches for comparative analysis of murine TCR repertoires, and show several examples of how these approaches can be applied for particular experimental settings. We analyze the efficiency of different metrics used for estimation of repertoire diversity, repertoires’ overlap, V- and J-gene segments usage similarity, and amino acid composition of CDR3. We discuss basic differences of these metrics, their advantages and limitations in different experimental models, and provide guidelines for choosing an efficient way to lead a comparative analysis of TCR repertoires. Applied to the various known and newly developed mouse models, such analysis should allow to disentangle multiple sophisticated puzzles in adaptive immunity.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Antigen receptor repertoire profiling from RNA-seq data. Nat Biotechnol.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Bolotin DA, Poslavsky S, Davydov AN, Frenkel FE, Fanchi L, Zolotareva OI, Hemmers S, Putintseva EV, Obraztsova AS, Shugay M, Ataullakhanov RI, Rudensky AY, Schumacher TN, Chudakov DM.</p><div class="notes publications__link">35(10):908-911 <a href="https://www.nature.com/articles/nbt.3979" class="notes--link" target="_blank">doi: 10.1038/nbt.3979.↗</a></div><div class="notes">2017 Oct 11</div></div><div class="publications__article"></div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">VDJdb: a curated database of T-cell receptor sequences with known antigen specificity. Nucleic Acids Res.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Shugay M, Bagaev DV, Zvyagin IV, Vroomans RM, Crawford JC, Dolton G, Komech EA, Sycheva AL, Koneva AE, Egorov ES, Eliseev AV, Van Dyk E, Dash P, Attaf M, Rius C, Ladell K, McLaren JE, Matthews KK, Clemens EB, Douek DC, Luciani F, van Baarle D, Kedzierska K, Kesmir C, Thomas PG, Price DA, Sewell AK, Chudakov DM.</p><div class="notes publications__link"><a href="https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkx760" class="notes--link" target="_blank">doi: 10.1093/nar/gkx760.↗</a></div><div class="notes">2017 Sep 1</div></div><div class="publications__article">The ability to decode antigen specificities encapsulated in the sequences of rearranged T-cell receptor (TCR) genes is critical for our understanding of the adaptive immune system and promises significant advances in the field of translational medicine. Recent developments in high-throughput sequencing methods (immune repertoire sequencing technology, or RepSeq) and single-cell RNA sequencing technology have allowed us to obtain huge numbers of TCR sequences from donor samples and link them to T-cell phenotypes. However, our ability to annotate these TCR sequences still lags behind, owing to the enormous diversity of the TCR repertoire and the scarcity of available data on T-cell specificities. In this paper, we present VDJdb, a database that stores and aggregates the results of published T-cell specificity assays and provides a universal platform that couples antigen specificities with TCR sequences. We demonstrate that VDJdb is a versatile instrument for the annotation of TCR repertoire data, enabling a concatenated view of antigen-specific TCR sequence motifs. VDJdb can be accessed at <a href="https://vdjdb.cdr3.net" class="notes--link" target="_blank">https://vdjdb.cdr3.net</a> and <a href="https://github.com/antigenomics/vdjdb-db" class="notes--link" target="_blank">https://github.com/antigenomics/vdjdb-db</a>.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Persisting fetal clonotypes influence the structure and overlap of adult human T cell receptor repertoires.PLoS Comput Biol.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Pogorelyy MV, Elhanati Y, Marcou Q, Sycheva AL, Komech EA, Nazarov VI, Britanova OV, Chudakov DM, Mamedov IZ, Lebedev YB, Mora T, Walczak AM.</p><div class="notes publications__link">13(7):e1005572. <a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005572" class="notes--link" target="_blank">doi: 10.1371/journal.pcbi.1005572.↗</a></div><div class="notes">2017 Jul 6</div></div><div class="publications__article">The diversity of T-cell receptors recognizing foreign pathogens is generated through a highly stochastic recombination process, making the independent production of the same sequence rare. Yet unrelated individuals do share receptors, which together constitute a "public" repertoire of abundant clonotypes. The TCR repertoire is initially formed prenatally, when the enzyme inserting random nucleotides is downregulated, producing a limited diversity subset. By statistically analyzing deep sequencing T-cell repertoire data from twins, unrelated individuals of various ages, and cord blood, we show that T-cell clones generated before birth persist and maintain high abundances in adult organisms for decades, slowly decaying with age. Our results suggest that large, low-diversity public clones are created during pre-natal life, and survive over long periods, providing the basis of the public repertoire.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">A high-throughput assay for quantitative measurement of PCR errors.Sci Rep.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Shagin DA, Shagina IA, Zaretsky AR, Barsova EV, Kelmanson IV, Lukyanov S, Chudakov DM, Shugay M</p><div class="notes publications__link">7(1):2718 <a href="https://www.nature.com/articles/s41598-017-02727-8" class="notes--link" target="_blank">doi: 10.1038/s41598-017-02727-8.↗</a></div><div class="notes">2017 Jun 2</div></div><div class="publications__article">The accuracy with which DNA polymerase can replicate a template DNA sequence is an extremely important property that can vary by an order of magnitude from one enzyme to another. The rate of nucleotide misincorporation is shaped by multiple factors, including PCR conditions and proofreading capabilities, and proper assessment of polymerase error rate is essential for a wide range of sensitive PCR-based assays. In this paper, we describe a method for studying polymerase errors with exceptional resolution, which combines unique molecular identifier tagging and high-throughput sequencing. Our protocol is less laborious than commonly-used methods, and is also scalable, robust and accurate. In a series of nine PCR assays, we have measured a range of polymerase accuracies that is in line with previous observations. However, we were also able to comprehensively describe individual errors introduced by each polymerase after either 20 PCR cycles or a linear amplification, revealing specific substitution preferences and the diversity of PCR error frequency profiles. We also demonstrate that the detected high-frequency PCR errors are highly recurrent and that the position in the template sequence and polymerase-specific substitution preferences are among the major factors influencing the observed PCR error rate.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Application of nonsense-mediated primer exclusion (NOPE) for preparation of unique molecular barcoded libraries. BMC Genomics.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Shagin DA, Turchaninova MA, Shagina IA, Shugay M, Zaretsky AR, Zueva OI, Bolotin DA, Lukyanov S, Chudakov DM.</p><div class="notes publications__link">BMC Genomics. 18(1):440. <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3815-2" class="notes--link" target="_blank">doi: 10.1186/s12864-017-3815-2.↗</a></div><div class="notes">2017 Jun 5</div></div><div class="publications__article">Recently we proposed efficient method to exclude undesirable primers at any stage of amplification reaction, here termed NOPE (NOnsense-mediated Primer Exclusion). According to this method, added oligonucleotide overlapping with the 3'-end of unwanted amplification primer (NOPE oligo) simultaneously provides a template for its elongation. This elongation disrupts specificity of unwanted primer, preventing its further participation in PCR. The suggested approach allows to rationally manage the course of PCR reactions in order to facilitate analysis of complex DNA mixtures as well as to perform multistage PCR bypassing intermediate purification steps. Here we apply NOPE method to DNA library preparation for the high-throughput sequencing (HTS) with the PCR-based introduction of unique molecular identifiers (UMI). We show that NOPE oligo efficiently neutralizes UMI-containing oligonucleotides after introduction of UMI into sample DNA molecules, thus allowing to proceed with further amplification steps without purification and associated loss of starting material. At the same time, NOPE oligo does not affect the efficiency of target PCR amplification. We describe a simple, robust and cheap modification of UMI-labeled HTS libraries preparation procedure, that allows to bypass purification step and thus to preserve starting material which may be limited, e.g. circulating tumor DNA, circulating fetal DNA, or small amounts of isolated cells of interest. Furthermore, demonstrated simplicity and robustness of NOPE method should make it popular in various PCR protocols.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">MAGERI: Computational pipeline for molecular-barcoded targeted resequencing.PLoS Comput Biol.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Shugay M, Zaretsky AR, Shagin DA, Shagina IA, Volchenkov IA, Shelenkov AA, Lebedin MY, Bagaev DV, Lukyanov S, Chudakov DM.</p><div class="notes publications__link">13(5):e1005480. <a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005480" class="notes--link" target="_blank">doi: 10.1371/journal.pcbi.1005480↗</a></div><div class="notes">2017 May 5</div></div><div class="publications__article">Unique molecular identifiers (UMIs) show outstanding performance in targeted high-throughput resequencing, being the most promising approach for the accurate identification of rare variants in complex DNA samples. This approach has application in multiple areas, including cancer diagnostics, thus demanding dedicated software and algorithms. Here we introduce MAGERI, a computational pipeline that efficiently handles all caveats of UMI-based analysis to obtain high-fidelity mutation profiles and call ultra-rare variants. Using an extensive set of benchmark datasets including gold-standard biological samples with known variant frequencies, cell-free DNA from tumor patient blood samples and publicly available UMI-encoded datasets we demonstrate that our method is both robust and efficient in calling rare variants. The versatility of our software is supported by accurate results obtained for both tumor DNA and viral RNA samples in datasets prepared using three different UMI-based protocols.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Clonal selection in the human Vδ1 T cell repertoire indicates γδ TCR-dependent adaptive immune surveillance. Nature Communications.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Davey MS, Willcox CR, Joyce SP, Ladell K, Kasatskaya SA, McLaren JE, Hunter S, Salim M, Mohammed F, Price DA, Chudakov DM, Willcox BE.</p><div class="notes publications__link">8:14760. <a href="https://www.nature.com/articles/ncomms14760" class="notes--link" target="_blank">doi:10.1038/ncomms14760.↗</a></div><div class="notes">2017 Mar 1</div></div><div class="publications__article">γδ T cells are considered to be innate-like lymphocytes that respond rapidly to stress without clonal selection and differentiation. Here we use next-generation sequencing to probe how this paradigm relates to human Vδ2neg T cells, implicated in responses to viral infection and cancer. The prevalent Vδ1 T cell receptor (TCR) repertoire is private and initially unfocused in cord blood, typically becoming strongly focused on a few high-frequency clonotypes by adulthood. Clonal expansions have differentiated from a naive to effector phenotype associated with CD27 downregulation, retaining proliferative capacity and TCR sensitivity, displaying increased cytotoxic markers and altered homing capabilities, and remaining relatively stable over time. Contrastingly, Vδ2+ T cells express semi-invariant TCRs, which are present at birth and shared between individuals. Human Vδ1+ T cells have therefore evolved a distinct biology from the Vδ2+ subset, involving a central, personalized role for the γδ TCR in directing a highly adaptive yet unconventional form of immune surveillance.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="h2">2016</div><div class="publications__list-item-inner"><p class="h3">Regulatory T Cells Exhibit Distinct Features in Human Breast Cancer. Immunity.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Plitas G, Konopacki C, Wu K, Bos PD, Morrow M, Putintseva EV, Chudakov DM, Rudensky AY.</p><div class="notes publications__link">45(5):1122-1134.  <a href="http://www.cell.com/immunity/abstract/S1074-7613(16)30443-5?_returnURL=http%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1074761316304435%3Fshowall%3Dtrue" class="notes--link" target="_blank">doi: 10.1016/j.immuni.2016.10.032.↗</a></div><div class="notes">2016 Nov 15</div></div><div class="publications__article">Regulatory T (Treg) cells reside in lymphoid organs and barrier tissues where they control different types of inflammatory responses. Treg cells are also found in human cancers, and studies in animal models suggest that they contribute to cancer progression. However, properties of human intratumoral Treg cells and those present in corresponding normal tissue remain largely unknown. Here, we analyzed features of Treg cells in untreated human breast carcinomas, normal mammary gland, and peripheral blood. Tumor-resident Treg cells were potently suppressive and their gene-expression pattern resembled that of normal breast tissue, but not of activated peripheral blood Treg cells. Nevertheless, a number of cytokine and chemokine receptor genes, most notably CCR8, were upregulated in tumor-resident Treg cells in comparison to normal tissue-resident ones. Our studies suggest that targeting CCR8 for the depletion of tumor-resident Treg cells might represent a promising immunotherapeutic approach for the treatment of breast cancer.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Tracking T-cell immune reconstitution after TCRαβ/CD19-depleted hematopoietic cells transplantation in children. Leukemia.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Zvyagin IV, Mamedov IZ, Tatarinova OV, Komech EA, Kurnikova EE, Boyakova EV, Brilliantova V, Shelikhova LN, Balashov DN, Shugay M Sycheva AL, Kasatskaya SA, Lebedev YB, Maschan AA, Maschan MA, Chudakov DM.</p><div class="notes publications__link"> <a href="http://www.nature.com/leu/journal/vaop/ncurrent/full/leu2016321a.html" class="notes--link" target="_blank">doi:10.1038/leu.2016.321.↗</a></div><div class="notes">2016 Dec 9</div></div><div class="publications__article">αβT-cell-depleted allogeneic hematopoietic cell transplantation holds promise for the safe and accessible therapy of both malignant and non-malignant blood disorders. Here we employed molecular barcoding normalized T-cell receptor (TCR) profiling to quantitatively track T-cell immune reconstitution after TCRαβ-/CD19-depleted transplantation in children. We demonstrate that seemingly early reconstitution of αβT-cell counts 2 months after transplantation is based on only several hundred rapidly expanded clones originating from non-depleted graft cells. In further months, frequency of these hyperexpanded clones declines, and after 1 year the observed T-cell counts and TCRβ diversity are mostly provided by the newly produced T cells. We also demonstrate that high TCRβ diversity at day 60 observed for some of the patients is determined by recipient T cells and intrathymic progenitors that survived conditioning regimen. Our results indicate that further efforts on optimization of TCRαβ-/CD19-depleted transplantation protocols should be directed toward providing more efficient T-cell defense in the first months after transplantation.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">High-quality full-length immunoglobulin profiling with unique molecular barcoding. Nature Protocols<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Turchaninova MA, Davydov A, Britanova OV, Shugay M, Bikos V, Egorov ES, Kirgizova VI, Merzlyak EM, Staroverov DB, Bolotin DA, Mamedov IZ, Izraelson M, Logacheva MD, Kladova O, Plevova K, Pospisilova S &amp; Chudakov DM.</p><div class="notes publications__link">11(9):1599-616. <a href="http://www.nature.com/nprot/journal/v11/n9/full/nprot.2016.093.html" class="notes--link" target="_blank">doi: 10.1038/nprot.2016.093.↗</a></div><div class="notes">2016 Sep</div></div><div class="publications__article">High-throughput sequencing analysis of hypermutating immunoglobulin (IG) repertoires remains a challenging task. Here we present a robust protocol for the full-length profiling of human and mouse IG repertoires. This protocol uses unique molecular identifiers (UMIs) introduced in the course of cDNA synthesis to control bottlenecks and to eliminate PCR and sequencing errors. Using asymmetric 400+100-nt paired-end Illumina sequencing and UMI-based assembly with the new version of the MIGEC software, the protocol allows up to 750-nt lengths to be sequenced in an almost error-free manner. This sequencing approach should also be applicable to various tasks beyond immune repertoire studies. In IG profiling, the achieved length of high-quality sequence covers the variable region of even the longest chains, along with the fragment of a constant region carrying information on the antibody isotype. The whole protocol, including preparation of cells and libraries, sequencing and data analysis, takes 5 to 6 d.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Single-cell analysis of glandular T cell receptors in Sjögren's syndrome. JCI Insight.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Joachims ML, Leehan KM, Lawrence C, Pelikan RC, Moore JS, Pan Z, Rasmussen A, Radfar L, Lewis DM, Grundahl KM, Kelly JA, Wiley GB, Shugay M, Chudakov DM, Lessard CJ, Stone DU, Scofield RH, Montgomery CG, Sivils KL, Thompson LF, Farris AD.</p><div class="notes publications__link">1(8). pii: e85609. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4922426/" class="notes--link" target="_blank">doi: 10.1172/jci.insight.85609.↗</a></div><div class="notes">2016 Jun 2</div></div><div class="publications__article">CD4+ T cells predominate in salivary gland (SG) inflammatory lesions in Sjögren’s syndrome (SS). However, their antigen specificity, degree of clonal expansion, and relationship to clinical disease features remain unknown. We used multiplex reverse-transcriptase PCR to amplify paired T cell receptor α (TCRα) and β transcripts of single CD4+CD45RA– T cells from SG and peripheral blood (PB) of 10 individuals with primary SS, 9 of whom shared the HLA DR3/DQ2 risk haplotype. TCRα and β sequences were obtained from a median of 91 SG and 107 PB cells per subject. The degree of clonal expansion and frequency of cells expressing two productively rearranged α genes were increased in SG versus PB. Expanded clones from SG exhibited complementary-determining region 3 (CDR3) sequence similarity both within and among subjects, suggesting antigenic selection and shared antigen recognition. CDR3 similarities were shared among expanded clones from individuals discordant for canonical Ro and La autoantibodies, suggesting recognition of alternative SG antigen(s). The extent of SG clonal expansion correlated with reduced saliva production and increased SG fibrosis, linking expanded SG T cells with glandular dysfunction. Knowledge of paired TCRα and β sequences enables further work toward identification of target antigens and development of novel therapies.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">VDJviz: a versatile browser for immunogenomics data. BMC Genomics.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Bagaev DV, Zvyagin IV, Putintseva EV, Izraelson M, Britanova OV, Chudakov DM, Shugay M.</p><div class="notes publications__link">17(1):453. <a href="http://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-2799-7" class="notes--link" target="_blank">doi: 10.1186/s12864-016-2799-7.↗</a></div><div class="notes">2016 Jun 13</div></div><div class="publications__article">Background<br>
The repertoire of T- and B-cell receptor sequences encodes the antigen specificity of adaptive immunity system, determines its present state and guides its ability to mount effective response against encountered antigens in future. High throughput sequencing of immune repertoires (Rep-Seq) is a promising technique that allows to profile millions of antigen receptors of an individual in a single experiment. While a substantial number of tools for mapping and assembling Rep-Seq data were published recently, the field still lacks an intuitive and flexible tool that can be used by researchers with little or no computational background for in-depth analysis of immune repertoire profiles.<br><br>Results<br>
Here we report VDJviz, a web tool that can be used to browse, analyze and perform quality control of Rep-Seq results generated by various pre-processing software. On a set of real data examples we show that VDJviz can be used to explore key repertoire characteristics such as spectratype, repertoire clonality, V-(D)-J recombination patterns and to identify shared clonotypes. We also demonstrate the utility of VDJviz in detection of critical Rep-Seq biases such as artificial repertoire diversity and cross-sample contamination.<br><br>Conclusions<br>
VDJviz is a versatile and lightweight tool that can be easily employed by biologists, immunologists and immunogeneticists for routine analysis and quality control of Rep-Seq data. The software is freely available for non-commercial purposes, and can be downloaded from: https://github.com/antigenomics/vdjviz.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Dynamics of Individual T Cell Repertoires: From Cord Blood to Centenarians. J Immunol.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Britanova OV, Shugay M, Merzlyak EM, Staroverov DB, Putintseva EV, Turchaninova MA, Mamedov IZ, Pogorelyy MV, Bolotin DA, Izraelson M, Davydov AN, Egorov ES, Kasatskaya SA, Rebrikov DV, Lukyanov S, Chudakov DM.</p><div class="notes publications__link">196(12):5005-13. <a href="http://www.jimmunol.org/content/196/12/5005.long" class="notes--link" target="_blank"> doi: 10.4049/jimmunol.1600005.↗</a></div><div class="notes">2016 Jun 15</div></div><div class="publications__article">The diversity, architecture, and dynamics of the TCR repertoire largely determine our ability to effectively withstand infections and malignancies with minimal mistargeting of immune responses. In this study, we have employed deep TCRβ repertoire sequencing with normalization based on unique molecular identifiers to explore the long-term dynamics of T cell immunity. We demonstrate remarkable stability of repertoire, where approximately half of all T cells in peripheral blood are represented by clones that persist and generally preserve their frequencies for 3 y. We further characterize the extremes of lifelong TCR repertoire evolution, analyzing samples ranging from umbilical cord blood to centenarian peripheral blood. We show that the fetal TCR repertoire, albeit structurally maintained within regulated borders due to the lower numbers of randomly added nucleotides, is not limited with respect to observed functional diversity. We reveal decreased efficiency of nonsense-mediated mRNA decay in umbilical cord blood, which may reflect specific regulatory mechanisms in development. Furthermore, we demonstrate that human TCR repertoires are functionally more similar at birth but diverge during life, and we track the lifelong behavior of CMV- and EBV-specific T cell clonotypes. Finally, we reveal gender differences in dynamics of TCR diversity constriction, which come to naught in the oldest age. Based on our data, we propose a more general explanation for the previous observations on the relationships between longevity and immunity.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">CR usage, gene expression and function of two distinct FOXP3(+)Treg subsets within CD4(+)CD25(hi) T cells identified by expression of CD39 and CD45RO. Immunol Cell Biol.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Ye L, Goodall JC, Zhang L, Putintseva EV, Lam B, Jiang L, Liu W, Yin J, Lin L, Li T, Wu X, Yeo G, Shugay M, Chudakov DM, Gaston H, Xu H.</p><div class="notes publications__link">94(3):293-305. <a href="http://www.nature.com/icb/journal/v94/n3/full/icb201590a.html" class="notes--link" target="_blank"> doi: 10.1038/icb.2015.90.↗</a></div><div class="notes">2016 Mar</div></div><div class="publications__article">FOXP3+ regulatory T (Treg) cells are indispensable for immune homeostasis, but their study in humans is complicated by heterogeneity within Treg, the difficulty in purifying Tregs using surface marker expression (e.g. CD25) and the transient expression of FOXP3 by activated effector cells. Here, we report that expression of CD39 and CD45RO distinguishes three sub-populations within human CD4+CD25hi T cells. Initial phenotypic and functional analysis demonstrated that CD4+CD25hiCD39+CD45RO+ cells had properties consistent with effector Treg, CD4+CD25hiCD39−CD45RO− cells were naïve Treg and CD4+CD25hiCD39−CD45RO+ cells were predominantly non-Treg with effector T-cell function. Differences in these two newly identified Treg subsets were corroborated by studies of gene expression and TCR analysis. To apply this approach, we studied these two newly identified Treg subsets in ankylosing spondylitis, and showed impairment in both effector and naïve Treg. This work highlights the importance of discriminating Treg subsets to enable proper comparisons of immune regulatory capacity in healthy individuals and those with inflammatory disease.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="h2">2015</div><div class="publications__list-item-inner"><p class="h3">Sequencing rare T-cell populations. Oncotarget.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Shugay M, Lukyanov S, Chudakov DM.</p><div class="notes publications__link">6(37):39393-4. <a href="http://www.impactjournals.com/oncotarget/index.php?journal=oncotarget&page=article&op=view&path[]=6349&pubmed-linkout=1" class="notes--link" target="_blank"> doi: 10.18632/oncotarget.6349.↗</a></div><div class="notes">2015 Nov 24</div></div><div class="publications__article">High-throughput sequencing of T cell (TCR) receptor repertoires is a promising approach that can be used to characterize the state and dynamics of adaptive immunity and, potentially, to deduce the antigen specificityof the immune response.<br><br>While the number of applications of TCR profiling is constantly growing, basic protocols have several drawbacks that are making it difficult to utilize them in certain experimental settings. In particular, quantitativedata interpretation is hampered by stochasticity ofsampling of lymphocytes and TCR-encoding RNA/DNA, stochasticity and biases of PCR amplification, andsequencing quality biases.<br><br>These problems become more evident when profiling rare T-cell subpopulations, such as tumor- infiltrating lymphocytes (TILs) that may contain low lymphocyte counts, populations enriched for tumor antigen-specific cells, or functional T cell subests of interest, the tasks critical for studying the role of adaptive immunity in cancer progression and treatment.<br><br>In this brief note we summarize what we suggest to be the most relevant issues of studying tumor-specific TCR repertoire for the minor lymphocyte counts, where derived data is more prone to noise introduced by stochastic sampling, which renders methods that operate with T-cell clonotype frequencies less useful.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">VDJtools: Unifying Post-analysis of T Cell Receptor Repertoires. PLoS Comput Biol.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Shugay M, Bagaev DV, Turchaninova MA, Bolotin DA, Britanova OV, Putintseva EV, Pogorelyy MV, Nazarov VI, Zvyagin IV, Kirgizova VI, Kirgizov KI, Skorobogatova EV, Chudakov DM.</p><div class="notes publications__link">11(11). <a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004503" class="notes--link" target="_blank"> doi: 10.1371/journal.pcbi.1004503.↗</a></div><div class="notes">2015 Nov 25</div></div><div class="publications__article">Despite the growing number of immune repertoire sequencing studies, the field still lacks software for analysis and comprehension of this high-dimensional data. Here we report VDJtools, a complementary software suite that solves a wide range of T cell receptor (TCR) repertoires post-analysis tasks, provides a detailed tabular output and publication-ready graphics, and is built on top of a flexible API. Using TCR datasets for a large cohort of unrelated healthy donors, twins, and multiple sclerosis patients we demonstrate that VDJtools greatly facilitates the analysis and leads to sound biological conclusions. VDJtools software and documentation are available at <a href="https://github.com/mikessh/vdjtools" class="notes--link" target="_blank">https://github.com/mikessh/vdjtools.</a></div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">A mechanism for expansion of regulatory T-cell repertoire and its role in self-tolerance. Nature.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Feng Y, van der Veeken J, Shugay M, Putintseva EV, Osmanbeyoglu HU, Dikiy S, Hoyos BE, Moltedo B, Hemmers S, Treuting P, Leslie CS, Chudakov DM, Rudensky AY.</p><div class="notes publications__link">528(7580):132-6. <a href="http://www.nature.com/nature/journal/v528/n7580/full/nature16141.html" class="notes--link" target="_blank"> doi: 10.1038/nature16141.↗</a></div><div class="notes">2015 Dec 3</div></div><div class="publications__article">T-cell receptor (TCR) signalling has a key role in determining T-cell fate. Precursor cells expressing TCRs within a certain low-affinity range for complexes of self-peptide and major histocompatibility complex (MHC) undergo positive selection and differentiate into naive T cells expressing a highly diverse self-MHC-restricted TCR repertoire. In contrast, precursors displaying TCRs with a high affinity for ‘self’ are either eliminated through TCR-agonist-induced apoptosis (negative selection) or restrained by regulatory T (Treg) cells, whose differentiation and function are controlled by the X-chromosome-encoded transcription factor Foxp3. Foxp3 is expressed in a fraction of self-reactive T cells that escape negative selection in response to agonist-driven TCR signals combined with interleukin 2 (IL-2) receptor signalling. In addition to Treg cells, TCR-agonist-driven selection results in the generation of several other specialized T-cell lineages such as natural killer T cells and innate mucosal-associated invariant T cells. Although the latter exhibit a restricted TCR repertoire, Treg cells display a highly diverse collection of TCRs. Here we explore in mice whether a specialized mechanism enables agonist-driven selection of Treg cells with a diverse TCR repertoire, and the importance this holds for self-tolerance. We show that the intronic Foxp3 enhancer conserved noncoding sequence 3 (CNS3) acts as an epigenetic switch that confers a poised state to the Foxp3 promoter in precursor cells to make Treg cell lineage commitment responsive to a broad range of TCR stimuli, particularly to suboptimal ones. CNS3-dependent expansion of the TCR repertoire enables Treg cells to control self-reactive T cells effectively, especially when thymic negative selection is genetically impaired. Our findings highlight the complementary roles of these two main mechanisms of self-tolerance.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">tcR: an R package for T cell receptor repertoire advanced data analysis. BMC Bioinformatics.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Nazarov VI, Pogorelyy MV, Komech EA, Zvyagin IV, Bolotin DA, Shugay M, Chudakov DM, Lebedev YB, Mamedov IZ.</p><div class="notes publications__link">16:175. <a href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0613-1" class="notes--link" target="_blank"> doi: 10.1186/s12859-015-0613-1.↗</a></div><div class="notes">2015 May 28</div></div><div class="publications__article">Background<br>The Immunoglobulins (IG) and the T cell receptors (TR) play the key role in antigen recognition during the adaptive immune response. Recent progress in next-generation sequencing technologies has provided an opportunity for the deep T cell receptor repertoire profiling. However, a specialised software is required for the rational analysis of massive data generated by next-generation sequencing.<br><br>Results<br>Here we introduce tcR, a new R package, representing a platform for the advanced analysis of T cell receptor repertoires, which includes diversity measures, shared T cell receptor sequences identification, gene usage statistics computation and other widely used methods. The tool has proven its utility in recent research studies.<br><br>Conclusions<br>tcR is an R package for the advanced analysis of T cell receptor repertoires after primary TR sequences extraction from raw sequencing reads. The stable version can be directly installed from The Comprehensive R Archive Network (http://cran.r-project.org/mirrors.html). The source code and development version are available at tcR GitHub (http://imminfo.github.io/tcr/) along with the full documentation and typical usage examples.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Quantitative profiling of immune repertoires for minor lymphocyte counts using unique molecular identifiers. J Immunol.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Egorov ES, Merzlyak EM, Shelenkov AA, Britanova OV, Sharonov GV, Staroverov DB, Bolotin DA, Davydov AN, Barsova E, Lebedev YB, Shugay M, Chudakov DM.</p><div class="notes publications__link">194(12):6155-63. <a href="http://www.jimmunol.org/content/194/12/6155.long" class="notes--link" target="_blank">doi: 10.4049/jimmunol.1500215.↗</a></div><div class="notes">2015 Jun 15</div></div><div class="publications__article">Emerging high-throughput sequencing methods for the analyses of complex structure of TCR and BCR repertoires give a powerful impulse to adaptive immunity studies. However, there are still essential technical obstacles for performing a truly quantitative analysis. Specifically, it remains challenging to obtain comprehensive information on the clonal composition of small lymphocyte populations, such as Ag-specific, functional, or tissue-resident cell subsets isolated by sorting, microdissection, or fine needle aspirates. In this study, we report a robust approach based on unique molecular identifiers that allows profiling Ag receptors for several hundred to thousand lymphocytes while preserving qualitative and quantitative information on clonal composition of the sample. We also describe several general features regarding the data analysis with unique molecular identifiers that are critical for accurate counting of starting molecules in high-throughput sequencing applications.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">MiXCR: software for comprehensive adaptive immunity profiling. Nat Methods.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Bolotin DA, Poslavsky S, Mitrophanov I, Shugay M, Mamedov IZ, Putintseva EV, Chudakov DM.</p><div class="notes publications__link">12(5):380-1. <a href="http://www.nature.com/nmeth/journal/v12/n5/full/nmeth.3364.html" class="notes--link" target="_blank">doi: 10.1038/nmeth.3364.↗</a></div><div class="notes">2015 May</div></div><div class="publications__article">High-throughput sequencing is gaining importance in adaptive immunity studies, demanding efficient software solutions for immunoglobulin (IG) and T-cell receptor profiling. Here we report MiXCR (available at http://mixcr.milaboratory.com/ and https://github.com/milaboratory/mixcr/), a universal framework that processes big immunome data from raw sequences to quantitated clonotypes. MiXCR efficiently handles paired- and single-end reads, considers sequence quality, corrects PCR errors and identifies germline hypermutations. The software supports both partial- and full-length profiling and employs all available RNA or DNA information, including sequences upstream of V and downstream of J gene segments.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="h2">2014</div><div class="publications__list-item-inner"><p class="h3">Towards error-free profiling of immune repertoires. Nat Methods.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Shugay M, Britanova OV, Merzlyak EM, Turchaninova MA, Mamedov IZ, Tuganbaev TR, Bolotin DA, Staroverov DB, Putintseva EV, Plevova K, Linnemann C, Shagin D, Pospisilova S, Lukyanov S, Schumacher TN, Chudakov DM.</p><div class="notes publications__link">Epub ahead of print. <a href="http://www.nature.com/nmeth/journal/v11/n6/full/nmeth.2960.html" class="notes--link" target="_blank">doi: 10.1038/nmeth.2960.↗</a></div><div class="notes">2014 May 4</div></div><div class="publications__article">Deep profiling of antibody and T cell–receptor repertoires by means of high-throughput sequencing has become an attractive approach for adaptive immunity studies, but its power is substantially compromised by the accumulation of PCR and sequencing errors. Here we report MIGEC (molecular identifier groups–based error correction), a strategy for high-throughput sequencing data analysis. MIGEC allows for nearly absolute error correction while fully preserving the natural diversity of complex immune repertoires.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Distinctive properties of identical twins’ TCR repertoires revealed by high-throughput sequencing. PNAS.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Zvyagin IV, Pogorely MV, Ivanova ME, Komech EA, Shugay M, Bolotin DA, Shelenkov AA, Kurnosov AA, Staroverova DB, Chudakov DM, Lebedev YB, Mamedov IZ.</p><div class="notes publications__link">Epub ahead of print. <a href="http://www.pnas.org/cgi/pmidlookup?view=long&pmid=24711416" class="notes--link" target="_blank">doi: 10.1073/pnas.1319389111.↗</a></div><div class="notes">2014 Apr 7</div></div><div class="publications__article">Adaptive immunity in humans is provided by hypervariable Ig-like molecules on the surface of B and T cells. The final set of these molecules in each organism is formed under the influence of two forces: individual genetic traits and the environment, which includes the diverse spectra of alien and self-antigens. Here we assess the impact of individual genetic factors on the formation of the adaptive immunity by analyzing the T-cell receptor (TCR) repertoires of three pairs of monozygous twins by next-generation sequencing. Surprisingly, we found that an overlap between the TCR repertoires of monozygous twins is similar to an overlap between the TCR repertoires of nonrelated individuals. However, the number of identical complementary determining region 3 sequences in two individuals is significantly increased for twin pairs in the fraction of highly abundant TCR molecules, which is enriched by the antigen-experienced T cells. We found that the initial recruitment of particular TCR V genes for recombination and subsequent selection in the thymus is strictly determined by individual genetic factors. J genes of TCRs are selected randomly for recombination; however, the subsequent selection in the thymus gives preference to some α but not β J segments. These findings provide a deeper insight into the mechanism of TCR repertoire generation.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Age-Related Decrease in TCR Repertoire Diversity Measured with Deep and Normalized Sequence Profiling. J Immunol.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Britanova OV, Putintseva EV, Shugay M, Merzlyak EM, Turchaninova MA, Staroverov DB, Bolotin DA, Lukyanov S, Bogdanova EA, Mamedov IZ, Lebedev YB, Chudakov DM.</p><div class="notes publications__link">Epub ahead of print. <a href="http://www.jimmunol.org/cgi/pmidlookup?view=long&pmid=24510963" class="notes--link" target="_blank">doi: 10.4049/jimmunol.1302064.↗</a></div><div class="notes">2014 Dec 7</div></div><div class="publications__article">The decrease of TCR diversity with aging has never been studied by direct methods. In this study, we combined high-throughput Illumina sequencing with unique cDNA molecular identifier technology to achieve deep and precisely normalized profiling of TCR β repertoires in 39 healthy donors aged 6–90 y. We demonstrate that TCR β diversity per 106 T cells decreases roughly linearly with age, with significant reduction already apparent by age 40. The percentage of naive T cells showed a strong correlation with measured TCR diversity and decreased linearly up to age 70. Remarkably, the oldest group (average age 82 y) was characterized by a higher percentage of naive CD4+ T cells, lower abundance of expanded clones, and increased TCR diversity compared with the previous age group (average age 62 y), suggesting the influence of age selection and association of these three related parameters with longevity. Interestingly, cross-analysis of individual TCR β repertoires revealed a set >10,000 of the most representative public TCR β clonotypes, whose abundance among the top 100,000 clones correlated with TCR diversity and decreased with aging.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="h2">2013</div><div class="publications__list-item-inner"><p class="h3">Huge Overlap of Individual TCR Beta Repertoires.Front Immunol.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Shugay M, Bolotin DA, Putintseva EV, Pogorelyy MV, Mamedov IZ, Chudakov DM.</p><div class="notes publications__link">4:466. <a href="http://journal.frontiersin.org/article/10.3389/fimmu.2013.00466/full" class="notes--link" target="_blank">doi: 10.3389/fimmu.2013.00466.↗</a></div><div class="notes">2013 Dec 25</div></div><div class="publications__article">It has been reported that human TCR repertoires commonly carry so-called public clonotypes – CDR3 variants that are often shared between individuals. Cross-comparison of individual immune repertoires has previously revealed the existence of a population of TCR beta CDR3 variants that are identical at the amino acid level for any two donors. The lower bound for the total overlap between any two given donors’ TCR beta repertoires within their CD8+ naïve T cell subset has been estimated as ~14,000 identical amino acid CDR3 variants based on comparison of 200,000–600,000 individual TCR beta clonotypes. Here, we have used deep profiling data consisting of 1–2 × 10-6 individual TCR beta clonotypes that we obtained from healthy donors to better estimate the total overlap between TCR beta repertoires for any two individuals.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Mother and child T cell receptor repertoires: deep profiling study. Front Immunol.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Putintseva EV, Britanova OV, Staroverov DB, Merzlyak EM, Turchaninova MA, Shugay M, Bolotin DA, Pogorelyy MV, Mamedov IZ, Bobrynina V, Maschan M, Lebedev YB, Chudakov DM.</p><div class="notes publications__link">4:463. <a href="http://journal.frontiersin.org/article/10.3389/fimmu.2013.00463/abstract" class="notes--link" target="_blank">doi: 10.3389/fimmu.2013.00463.↗</a></div><div class="notes">2013 Dec 25</div></div><div class="publications__article">The relationship between maternal and child immunity has been actively studied in the context of complications during pregnancy, autoimmune diseases, and haploidentical transplantation of hematopoietic stem cells and solid organs. Here, we have for the first time used high-throughput Illumina HiSeq sequencing to perform deep quantitative profiling of T cell receptor (TCR) repertoires for peripheral blood samples of three mothers and their six children. Advanced technology allowed accurate identification of 5 × 105 to 2 × 106 TCR beta clonotypes per individual. We performed comparative analysis of these TCR repertoires with the aim of revealing characteristic features that distinguish related mother-child pairs, such as relative TCR beta variable segment usage frequency and relative overlap of TCR beta complementarity-determining region 3 (CDR3) repertoires. We show that thymic selection essentially and similarly shapes the initial output of the TCR recombination machinery in both related and unrelated pairs, with minor effect from inherited differences. The achieved depth of TCR profiling also allowed us to test the hypothesis that mature T cells transferred across the placenta during pregnancy can expand and persist as functional microchimeric clones in their new host, using characteristic TCR beta CDR3 variants as clonal identifiers.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Preparing unbiased T-cell receptor and antibody cDNA libraries for the deep next generation sequencing profiling. Front Immunol.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Mamedov IZ, Britanova OV, Zvyagin IV, Turchaninova MA, Bolotin DA, Putintseva EV, Lebedev YB, Chudakov DM.</p><div class="notes publications__link">4:456. <a href="http://journal.frontiersin.org/article/10.3389/fimmu.2013.00456/abstract" class="notes--link" target="_blank">doi: 10.3389/fimmu.2013.00456.↗</a></div><div class="notes">2013 Dec 23</div></div><div class="publications__article">High-throughput sequencing has the power to reveal the nature of adaptive immunity as represented by the full complexity of T-cell receptor (TCR) and antibody (IG) repertoires, but is at present severely compromised by the quantitative bias, bottlenecks, and accumulated errors that inevitably occur in the course of library preparation and sequencing. Here we report an optimized protocol for the unbiased preparation of TCR and IG cDNA libraries for high-throughput sequencing, starting from thousands or millions of live cells in an investigated sample. Critical points to control are revealed, along with tips that allow researchers to minimize quantitative bias, accumulated errors, and cross-sample contamination at each stage, and to enhance the subsequent bioinformatic analysis. The protocol is simple, reliable, and can be performed in 1–2 days.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">MiTCR: software for T-cell receptor sequencing data analysis. Nat Methods.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Bolotin DA, Shugay M, Mamedov IZ, Putintseva EV, Turchaninova MA, Zvyagin IV, Britanova OV, Chudakov DM.</p><div class="notes publications__link">10(9):813-4. <a href="http://www.nature.com/nmeth/journal/v10/n9/full/nmeth.2555.html" class="notes--link" target="_blank">doi: 10.1038/nmeth.2555.↗</a></div><div class="notes">2013 Aug</div></div><div class="publications__article">High-throughput sequencing technologies have transformed the field of antigen receptor diversity studies, enabling deep and quantitative analysis for deciphering adaptive immunity function in health and disease. As more data are produced each year, there is steadily growing demand for standardized analysis software.<br>Here we report MiTCR, an open-source software for rapid, robust and comprehensive analysis of hundreds of millions of raw high-throughput sequencing reads containing sequences encoding human or mouse α or β T-cell antigen receptor (TCR) chains (Supplementary Software). Raw data in FASTQ format generated via Illumina, 454 or Ion Torrent sequencing can be used as input for analysis. The only requirement is that sequence encoding conserved positions flanking the complementarity-determining region 3 (CDR3), Cys104 and Phe118 or Trp118, is covered by a sequencing read.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">Pairing of T-cell receptor chains via emulsion PCR. Eur J Immunol.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Turchaninova MA, Britanova OV, Bolotin DA, Shugay M, Putintseva EV, Staroverov DB, Sharonov G, Shcherbo D, Zvyagin IV, Mamedov IZ, Linnemann C, Schumacher TN, Chudakov DM.</p><div class="notes publications__link">43(9):2507-15. <a href="http://onlinelibrary.wiley.com/doi/10.1002/eji.201343453/abstract;jsessionid=9313B50324AE10EB5FE3F7B5C82A857E.f04t02?systemMessage=Wiley+Online+Library+will+have+be+unavailable+on+Saturday+5th+December+from+10%3A00-14%3A00+GMT+%2F+05%3A00-09%3A00+EST+%2F+18%3A00-22%3A00+SGT+for+essential+maintenance.+Apologies+for+the+inconvenience." class="notes--link" target="_blank">doi: 10.1002/eji.201343453.↗</a></div><div class="notes">2013 Sep</div></div><div class="publications__article">Our ability to analyze adaptive immunity and engineer its activity has long been constrained by our limited ability to identify native pairs of heavy–light antibody chains and alpha–beta T-cell receptor (TCR) chains — both of which comprise coupled “halves of a key”, collectively capable of recognizing specific antigens. Here, we report a cell-based emulsion RT-PCR approach that allows the selective fusion of the native pairs of amplified TCR alpha and beta chain genes for complex samples. A new type of PCR suppression technique was developed that makes it possible to amplify the fused library with minimal noise for subsequent analysis by high-throughput paired-end Illumina sequencing. With this technique, single analysis of a complex blood sample allows identification of multiple native TCR chain pairs. This approach may be extended to identify native antibody chain pairs and, more generally, pairs of mRNA molecules that are coexpressed in the same living cells.</div><button class="publications__more publications__more--mob" type="button"></button></div><div class="publications__list-item"><div class="publications__list-item-inner"><p class="h3">High-throughput identification of antigen-specific TCRs by TCR gene capture. Nat Medicine.<span class="publications__more publications__more--desktop" type="button"></span></p><p class="text">Linnemann, C., Heemskerk, B., Kvistborg, P., Kluin, R.J.C., Bolotin, D.A., Chen, X.,Bresser, K., Nieuwland, M., Schotte, R., Michels, M., Gomez-Eerland, R., Jahn, L., Hombrink, P., Legrand, N., Shu, C.J., Mamedov, I.Z., Velds, A., Blank, C.U., Haanen, J.B.A.G., Turchaninova, M.A., Kerkhoven, R.M., Spits, H., Reker Hadrup, S., Heemskerk, M.H.M, Blankenstein, T., Chudakov, D.M, Bendle, G.M. , Schumacher, T.N.M.</p><div class="notes publications__link">9(11):1534-41. <a href="http://www.nature.com/nm/journal/v19/n11/full/nm.3359.html" class="notes--link" target="_blank">doi: 10.1038/nm.3359.↗</a></div><div class="notes">2013 Nov</div></div><div class="publications__article">The transfer of T cell receptor (TCR) genes into patient T cells is a promising approach for the treatment of both viral infections and cancer. Although efficient methods exist to identify antibodies for the treatment of these diseases, comparable strategies to identify TCRs have been lacking. We have developed a high-throughput DNA-based strategy to identify TCR sequences by the capture and sequencing of genomic DNA fragments encoding the TCR genes. We establish the value of this approach by assembling a large library of cancer germline tumor antigen–reactive TCRs. Furthermore, by exploiting the quantitative nature of TCR gene capture, we show the feasibility of identifying antigen-specific TCRs in oligoclonal T cell populations from either human material or TCR-humanized mice. Finally, we demonstrate the ability to identify tumor-reactive TCRs within intratumoral T cell subsets without knowledge of antigen specificities, which may be the first step toward the development of autologous TCR gene therapy to target patient-specific neoantigens in human cancer.</div><button class="publications__more publications__more--mob" type="button"></button></div></div></div></main><footer class="footer"><div class="footer__container container"><div class="footer__left"><div class="footer__logos"><a class="footer__logo" href="./home.html"><svg width="207" height="43" viewBox="0 0 207 43" fill="none" xmlns="http://www.w3.org/2000/svg">
<path fill-rule="evenodd" clip-rule="evenodd" d="M0.569336 17.6421V0.322998H5.89438L9.91829 12.2329H9.96648L13.7735 0.322998H19.0985V17.6421H15.5565V5.36833H15.5083L11.2917 17.6421H8.37619L4.15952 5.48961H4.11133V17.6421H0.569336ZM22.4292 17.6421V0.322998H26.2121V17.6421H22.4292ZM0.569336 24.5794V41.8985H12.7374V38.6967H4.35229V24.5794H0.569336ZM21.4303 28.8486L19.1894 35.2038H23.6471L21.4785 28.8486H21.4303ZM13.0452 41.8985L19.5509 24.5794H23.4302L29.8636 41.8985H25.9361L24.635 38.0417H18.2015L16.8522 41.8985H13.0452ZM35.1302 31.5895V27.5387H38.6722C39.0095 27.5387 39.3308 27.571 39.636 27.6358C39.9572 27.6843 40.2384 27.7813 40.4793 27.9268C40.7203 28.0724 40.913 28.2745 41.0576 28.5332C41.2022 28.7919 41.2745 29.1234 41.2745 29.5278C41.2745 30.2554 41.0576 30.7809 40.6239 31.1044C40.1902 31.4278 39.636 31.5895 38.9613 31.5895H35.1302ZM31.3472 24.5794V41.8985H39.6842C40.4552 41.8985 41.2022 41.8015 41.925 41.6074C42.664 41.4134 43.3146 41.1142 43.8768 40.71C44.455 40.3056 44.9129 39.7882 45.2502 39.1575C45.5875 38.5107 45.7562 37.7507 45.7562 36.8774C45.7562 35.794 45.4911 34.8722 44.9611 34.1122C44.4471 33.336 43.6599 32.7942 42.5997 32.487C43.3708 32.115 43.949 31.6381 44.3346 31.0559C44.7362 30.4737 44.937 29.7461 44.937 28.8728C44.937 28.0642 44.8004 27.385 44.5273 26.8353C44.2703 26.2854 43.9009 25.8488 43.419 25.5254C42.937 25.1859 42.3588 24.9433 41.6841 24.7977C41.0094 24.6522 40.2625 24.5794 39.4432 24.5794H31.3472ZM35.1302 38.9392V34.185H39.2505C40.0697 34.185 40.7283 34.379 41.2263 34.7671C41.7243 35.139 41.9732 35.7697 41.9732 36.6591C41.9732 37.1118 41.893 37.4839 41.7323 37.7749C41.5877 38.066 41.387 38.3004 41.1299 38.4784C40.8729 38.64 40.5757 38.7613 40.2384 38.8422C39.9011 38.9068 39.5476 38.9392 39.1782 38.9392H35.1302ZM51.5896 31.1044C51.4291 31.8159 51.3487 32.5516 51.3487 33.3117C51.3487 34.0394 51.4291 34.7509 51.5896 35.4463C51.7664 36.1255 52.0395 36.74 52.4089 37.2898C52.7784 37.8395 53.2603 38.2843 53.8546 38.6239C54.449 38.9473 55.1718 39.109 56.0232 39.109C56.8746 39.109 57.5975 38.9473 58.1917 38.6239C58.7861 38.2843 59.268 37.8395 59.6375 37.2898C60.007 36.74 60.272 36.1255 60.4326 35.4463C60.6094 34.7509 60.6976 34.0394 60.6976 33.3117C60.6976 32.5516 60.6094 31.8159 60.4326 31.1044C60.272 30.3928 60.007 29.7622 59.6375 29.2124C59.268 28.6464 58.7861 28.2017 58.1917 27.8783C57.5975 27.5387 56.8746 27.3689 56.0232 27.3689C55.1718 27.3689 54.449 27.5387 53.8546 27.8783C53.2603 28.2017 52.7784 28.6464 52.4089 29.2124C52.0395 29.7622 51.7664 30.3928 51.5896 31.1044ZM48.144 36.8532C47.7585 35.7535 47.5657 34.5731 47.5657 33.3117C47.5657 32.018 47.7585 30.8214 48.144 29.7218C48.5296 28.606 49.0837 27.6357 49.8066 26.811C50.5295 25.9863 51.413 25.3394 52.4571 24.8705C53.5173 24.4015 54.706 24.1671 56.0232 24.1671C57.3565 24.1671 58.5452 24.4015 59.5893 24.8705C60.6335 25.3394 61.5169 25.9863 62.2397 26.811C62.9626 27.6357 63.5168 28.606 63.9023 29.7218C64.2878 30.8214 64.4806 32.018 64.4806 33.3117C64.4806 34.5731 64.2878 35.7535 63.9023 36.8532C63.5168 37.9366 62.9626 38.8826 62.2397 39.6912C61.5169 40.4997 60.6335 41.1384 59.5893 41.6074C58.5452 42.0602 57.3565 42.2866 56.0232 42.2866C54.706 42.2866 53.5173 42.0602 52.4571 41.6074C51.413 41.1384 50.5295 40.4997 49.8066 39.6912C49.0837 38.8826 48.5296 37.9366 48.144 36.8532ZM70.8495 32.4143V27.5387H74.9939C75.8613 27.5387 76.5119 27.7328 76.9456 28.1209C77.3793 28.4928 77.5962 29.0992 77.5962 29.9401C77.5962 30.8133 77.3793 31.444 76.9456 31.8321C76.5119 32.2202 75.8613 32.4143 74.9939 32.4143H70.8495ZM67.0666 24.5794V41.8985H70.8495V35.131H74.6325C75.5802 35.131 76.2628 35.3411 76.6806 35.7617C77.0982 36.1821 77.3712 36.845 77.4998 37.7507C77.5962 38.4459 77.6685 39.1736 77.7167 39.9338C77.7648 40.6937 77.8933 41.3487 78.1022 41.8985H81.8851C81.7085 41.656 81.5719 41.3649 81.4755 41.0253C81.3951 40.6695 81.3309 40.2976 81.2828 39.9095C81.2506 39.5214 81.2265 39.1413 81.2105 38.7694C81.1944 38.3974 81.1783 38.0741 81.1623 37.7992C81.1301 37.3626 81.0659 36.926 80.9695 36.4893C80.8891 36.0527 80.7527 35.6565 80.5599 35.3008C80.3671 34.9288 80.1181 34.6134 79.8129 34.3548C79.5077 34.0798 79.1222 33.8777 78.6564 33.7484V33.6999C79.6202 33.3118 80.3108 32.7457 80.7286 32.0019C81.1623 31.258 81.3791 30.3767 81.3791 29.358C81.3791 28.6949 81.2587 28.0804 81.0177 27.5145C80.7927 26.9323 80.4554 26.4229 80.0057 25.9863C79.572 25.5497 79.0419 25.2101 78.4154 24.9675C77.8049 24.7088 77.1143 24.5794 76.3432 24.5794H67.0666ZM91.0336 28.8486L88.7927 35.2038H93.2503L91.0817 28.8486H91.0336ZM82.6484 41.8985L89.1541 24.5794H93.0335L99.4669 41.8985H95.5394L94.2382 38.0417H87.8048L86.4555 41.8985H82.6484ZM104.758 27.7813V41.8985H108.54V27.7813H113.697V24.5794H99.6012V27.7813H104.758ZM118.957 31.1044C118.796 31.8159 118.716 32.5516 118.716 33.3117C118.716 34.0394 118.796 34.7509 118.957 35.4463C119.134 36.1255 119.408 36.74 119.777 37.2898C120.146 37.8395 120.628 38.2843 121.222 38.6239C121.817 38.9473 122.54 39.109 123.391 39.109C124.242 39.109 124.965 38.9473 125.56 38.6239C126.154 38.2843 126.636 37.8395 127.005 37.2898C127.374 36.74 127.639 36.1255 127.8 35.4463C127.977 34.7509 128.065 34.0394 128.065 33.3117C128.065 32.5516 127.977 31.8159 127.8 31.1044C127.639 30.3928 127.374 29.7622 127.005 29.2124C126.636 28.6464 126.154 28.2017 125.56 27.8783C124.965 27.5387 124.242 27.3689 123.391 27.3689C122.54 27.3689 121.817 27.5387 121.222 27.8783C120.628 28.2017 120.146 28.6464 119.777 29.2124C119.408 29.7622 119.134 30.3928 118.957 31.1044ZM115.512 36.8532C115.126 35.7535 114.934 34.5731 114.934 33.3117C114.934 32.018 115.126 30.8214 115.512 29.7218C115.897 28.606 116.451 27.6357 117.174 26.811C117.897 25.9863 118.781 25.3394 119.825 24.8705C120.885 24.4015 122.073 24.1671 123.391 24.1671C124.724 24.1671 125.913 24.4015 126.957 24.8705C128.001 25.3394 128.885 25.9863 129.607 26.811C130.33 27.6357 130.885 28.606 131.27 29.7218C131.656 30.8214 131.848 32.018 131.848 33.3117C131.848 34.5731 131.656 35.7535 131.27 36.8532C130.885 37.9366 130.33 38.8826 129.607 39.6912C128.885 40.4997 128.001 41.1384 126.957 41.6074C125.913 42.0602 124.724 42.2866 123.391 42.2866C122.073 42.2866 120.885 42.0602 119.825 41.6074C118.781 41.1384 117.897 40.4997 117.174 39.6912C116.451 38.8826 115.897 37.9366 115.512 36.8532ZM138.217 32.4143V27.5387H142.362C143.229 27.5387 143.88 27.7328 144.313 28.1209C144.747 28.4928 144.964 29.0992 144.964 29.9401C144.964 30.8133 144.747 31.444 144.313 31.8321C143.88 32.2202 143.229 32.4143 142.362 32.4143H138.217ZM134.434 24.5794V41.8985H138.217V35.131H142C142.947 35.131 143.631 35.3411 144.048 35.7617C144.465 36.1821 144.739 36.845 144.867 37.7507C144.964 38.4459 145.036 39.1736 145.084 39.9338C145.133 40.6937 145.261 41.3487 145.47 41.8985H149.253C149.076 41.656 148.94 41.3649 148.843 41.0253C148.763 40.6695 148.699 40.2976 148.65 39.9095C148.619 39.5214 148.595 39.1413 148.578 38.7694C148.562 38.3974 148.546 38.0741 148.53 37.7992C148.498 37.3626 148.434 36.926 148.337 36.4893C148.257 36.0527 148.12 35.6565 147.928 35.3008C147.735 34.9288 147.486 34.6134 147.181 34.3548C146.875 34.0798 146.49 33.8777 146.024 33.7484V33.6999C146.988 33.3118 147.679 32.7457 148.096 32.0019C148.53 31.258 148.747 30.3767 148.747 29.358C148.747 28.6949 148.626 28.0804 148.385 27.5145C148.161 26.9323 147.824 26.4229 147.373 25.9863C146.94 25.5497 146.41 25.2101 145.783 24.9675C145.173 24.7088 144.482 24.5794 143.711 24.5794H134.434ZM151.823 41.8985V24.5794H155.606V41.8985H151.823ZM158.929 24.5794V41.8985H171.989V38.6967H162.712V34.4518H171.049V31.4925H162.712V27.7813H171.796V24.5794H158.929ZM173.47 36.1497H177.132C177.132 36.7319 177.236 37.2251 177.445 37.6294C177.654 38.0336 177.927 38.3651 178.265 38.6239C178.618 38.8665 179.028 39.0524 179.493 39.1818C179.959 39.2949 180.441 39.3516 180.939 39.3516C181.276 39.3516 181.638 39.3273 182.023 39.2788C182.409 39.2141 182.77 39.1009 183.108 38.9392C183.445 38.7775 183.727 38.5592 183.951 38.2843C184.176 37.9932 184.288 37.6294 184.288 37.1928C184.288 36.7237 184.136 36.3438 183.831 36.0527C183.541 35.7617 183.156 35.5191 182.674 35.325C182.192 35.131 181.646 34.9612 181.035 34.8156C180.425 34.6701 179.807 34.5084 179.18 34.3305C178.537 34.1688 177.911 33.9747 177.301 33.7484C176.691 33.5058 176.144 33.1985 175.662 32.8266C175.18 32.4546 174.787 31.9937 174.482 31.444C174.192 30.8779 174.048 30.1988 174.048 29.4065C174.048 28.517 174.233 27.7489 174.602 27.1021C174.988 26.439 175.486 25.8893 176.096 25.4527C176.706 25.0161 177.397 24.6926 178.168 24.4824C178.939 24.2722 179.71 24.1671 180.481 24.1671C181.38 24.1671 182.24 24.2722 183.06 24.4824C183.895 24.6765 184.633 24.9998 185.276 25.4527C185.919 25.9054 186.425 26.4875 186.794 27.1991C187.18 27.8944 187.373 28.7434 187.373 29.7461H183.71C183.678 29.2285 183.565 28.8001 183.373 28.4605C183.196 28.1209 182.955 27.8541 182.65 27.66C182.344 27.466 181.992 27.3285 181.59 27.2476C181.204 27.1667 180.778 27.1264 180.313 27.1264C180.007 27.1264 179.703 27.1587 179.397 27.2234C179.091 27.288 178.811 27.4012 178.554 27.563C178.313 27.7246 178.112 27.9268 177.951 28.1694C177.79 28.412 177.71 28.7191 177.71 29.0911C177.71 29.4307 177.775 29.7056 177.903 29.9159C178.031 30.126 178.281 30.3201 178.65 30.498C179.036 30.6758 179.558 30.8537 180.216 31.0316C180.891 31.2095 181.766 31.4358 182.843 31.7108C183.164 31.7754 183.606 31.8967 184.168 32.0747C184.746 32.2363 185.317 32.5031 185.879 32.8751C186.44 33.247 186.922 33.7484 187.324 34.379C187.742 34.9935 187.951 35.7859 187.951 36.7562C187.951 37.5485 187.799 38.2843 187.493 38.9635C187.187 39.6427 186.73 40.2329 186.12 40.7342C185.525 41.2193 184.778 41.5993 183.879 41.8743C182.995 42.1491 181.968 42.2866 180.795 42.2866C179.847 42.2866 178.923 42.1653 178.024 41.9228C177.14 41.6963 176.353 41.3325 175.662 40.8312C174.988 40.3299 174.45 39.6912 174.048 38.915C173.646 38.1388 173.453 37.217 173.47 36.1497ZM195.672 39.4067C196.65 39.8183 197.695 40.0242 198.805 40.0242C199.914 40.0242 200.951 39.8183 201.915 39.4067C202.893 38.995 203.747 38.4217 204.478 37.6867C205.208 36.937 205.778 36.077 206.187 35.1067C206.61 34.1365 206.822 33.0927 206.822 31.9754C206.822 30.8581 206.61 29.8144 206.187 28.8442C205.778 27.8739 205.208 27.0212 204.478 26.2862C203.747 25.5511 202.893 24.9778 201.915 24.5662C200.951 24.1399 199.914 23.9267 198.805 23.9267C197.695 23.9267 196.65 24.1399 195.672 24.5662C194.708 24.9778 193.854 25.5511 193.109 26.2862C192.379 27.0212 191.809 27.8739 191.401 28.8442C190.992 29.8144 190.788 30.8581 190.788 31.9754C190.788 33.0927 190.992 34.1365 191.401 35.1067C191.809 36.077 192.379 36.937 193.109 37.6867C193.854 38.4217 194.708 38.995 195.672 39.4067ZM201.214 37.797C200.469 38.1204 199.667 38.2821 198.805 38.2821C197.928 38.2821 197.11 38.1204 196.351 37.797C195.607 37.4735 194.949 37.0252 194.38 36.4519C193.81 35.8785 193.365 35.2096 193.044 34.4452C192.723 33.6807 192.562 32.8575 192.562 31.9754C192.562 31.0934 192.723 30.2775 193.044 29.5278C193.365 28.7632 193.81 28.0944 194.38 27.5211C194.949 26.9478 195.607 26.4993 196.351 26.176C197.11 25.8525 197.928 25.6908 198.805 25.6908C199.667 25.6908 200.469 25.8525 201.214 26.176C201.973 26.4993 202.638 26.9478 203.207 27.5211C203.777 28.0944 204.223 28.7632 204.544 29.5278C204.865 30.2775 205.026 31.0934 205.026 31.9754C205.026 32.8575 204.865 33.6807 204.544 34.4452C204.223 35.2096 203.777 35.8785 203.207 36.4519C202.638 37.0252 201.973 37.4735 201.214 37.797ZM195.628 27.8519V36.1211H197.731V33.6954H198.98L199.834 35.1949C199.995 35.4595 200.207 35.6801 200.469 35.8565C200.746 36.0329 201.068 36.1211 201.433 36.1211H202.66V34.357H201.871C201.769 34.357 201.674 34.3055 201.587 34.2026L200.951 33.1442C201.36 32.8942 201.682 32.5634 201.915 32.1519C202.164 31.7255 202.288 31.2478 202.288 30.7185C202.288 30.1599 202.148 29.6673 201.871 29.2411C201.608 28.8147 201.25 28.4766 200.798 28.2267C200.36 27.9768 199.87 27.8519 199.33 27.8519H195.628ZM199.155 31.8211H197.731V29.7483H199.155C199.418 29.7483 199.651 29.8365 199.856 30.0129C200.075 30.1893 200.185 30.4245 200.185 30.7185C200.185 31.086 200.083 31.3653 199.878 31.5565C199.673 31.7329 199.432 31.8211 199.155 31.8211Z" fill="#0F0E12"/>
</svg>
</a><a class="footer__logo-sk" href="#"><svg width="80" height="60" viewBox="0 0 80 60" fill="none" xmlns="http://www.w3.org/2000/svg">
<rect width="80" height="60" fill="#0F0E12"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M34.4582 42.0833V40.6953H32.9931V42.0833H34.4582ZM11.9072 49.8712V46.0929H14.2719C14.6403 46.0929 14.9423 46.14 15.1779 46.2343C15.4135 46.3285 15.5998 46.4763 15.7369 46.6776C15.874 46.879 15.9682 47.1317 16.0197 47.4359C16.0711 47.74 16.0968 48.102 16.0968 48.5218C16.0968 48.7446 16.1075 48.9759 16.1289 49.2158C16.1503 49.4557 16.221 49.6742 16.3409 49.8712H18.063C17.8917 49.6827 17.7653 49.3679 17.6839 48.9266C17.6025 48.4854 17.5618 47.9949 17.5618 47.4552C17.5618 47.0953 17.5168 46.7954 17.4269 46.5556C17.3369 46.3157 17.2234 46.1208 17.0863 45.9708C16.9492 45.8209 16.8014 45.7116 16.6429 45.6431C16.4844 45.5746 16.3367 45.5317 16.1996 45.5146V45.4889C16.2767 45.4632 16.3988 45.4118 16.5658 45.3347C16.7329 45.2576 16.9021 45.1312 17.0735 44.9556C17.2448 44.7799 17.3948 44.5486 17.5233 44.2616C17.6518 43.9746 17.716 43.6126 17.716 43.1756C17.716 42.396 17.4633 41.7877 16.9578 41.3507C16.4523 40.9138 15.6941 40.6953 14.6831 40.6953H10.3008V49.8712H11.9072ZM14.5297 44.8728H11.8951V42.0069H14.504C14.7096 42.0069 14.9088 42.024 15.1016 42.0583C15.2943 42.0926 15.4657 42.1611 15.6156 42.2639C15.7655 42.3667 15.8834 42.5103 15.969 42.6945C16.0547 42.8787 16.0975 43.1164 16.0975 43.4077C16.0975 43.8961 15.9647 44.2623 15.6992 44.5065C15.4336 44.7507 15.0437 44.8728 14.5297 44.8728ZM23.1072 49.9031C22.7774 50.0016 22.4325 50.0509 22.0727 50.0509C21.5501 50.0509 21.0874 49.9652 20.6847 49.7939C20.2821 49.6225 19.9415 49.3826 19.6631 49.0742C19.3846 48.7658 19.1747 48.3973 19.0333 47.969C18.892 47.5406 18.8213 47.0694 18.8213 46.5553C18.8213 46.0841 18.8963 45.6364 19.0462 45.2123C19.1961 44.7882 19.4103 44.4156 19.6888 44.0943C19.9672 43.773 20.3035 43.5181 20.6976 43.3296C21.0917 43.1411 21.5372 43.0469 22.0341 43.0469C22.5568 43.0469 23.0258 43.1561 23.4414 43.3746C23.8569 43.5931 24.2017 43.8822 24.4759 44.2421C24.7501 44.6019 24.9493 45.0153 25.0735 45.4822C25.1977 45.9492 25.2299 46.4311 25.1699 46.928H20.2863C20.2863 47.185 20.3228 47.4335 20.3956 47.6734C20.4684 47.9133 20.5776 48.1232 20.7233 48.3031C20.8689 48.483 21.0531 48.6265 21.2759 48.7336C21.4987 48.8407 21.7643 48.8943 22.0727 48.8943C22.5011 48.8943 22.8459 48.8022 23.1072 48.618C23.3685 48.4338 23.5635 48.1575 23.692 47.789H25.0799C25.0028 48.1489 24.87 48.4702 24.6815 48.7529C24.493 49.0356 24.266 49.2734 24.0004 49.4662C23.7348 49.6589 23.4371 49.8046 23.1072 49.9031ZM23.7036 45.9641H20.2852C20.2937 45.7157 20.343 45.4844 20.4329 45.2702C20.5229 45.056 20.6429 44.8696 20.7928 44.7111C20.9427 44.5526 21.1226 44.4284 21.3325 44.3384C21.5425 44.2485 21.7759 44.2035 22.0329 44.2035C22.2814 44.2035 22.5063 44.2528 22.7076 44.3513C22.909 44.4498 23.0803 44.5783 23.2217 44.7368C23.3631 44.8953 23.4744 45.0817 23.5558 45.2959C23.6372 45.5101 23.6865 45.7328 23.7036 45.9641ZM29.9136 49.9352C29.5624 50.0123 29.2154 50.0509 28.8727 50.0509C28.4528 50.0509 28.0652 50.0038 27.7096 49.9095C27.3541 49.8153 27.0456 49.6718 26.7843 49.479C26.523 49.2862 26.3174 49.0463 26.1674 48.7593C26.0175 48.4723 25.9383 48.1317 25.9297 47.7376H27.3947C27.4376 48.166 27.6004 48.4659 27.8831 48.6372C28.1658 48.8086 28.5043 48.8943 28.8984 48.8943C29.0354 48.8943 29.1918 48.8836 29.3674 48.8621C29.5431 48.8407 29.708 48.8 29.8622 48.74C30.0164 48.6801 30.1428 48.5923 30.2413 48.4766C30.3399 48.3609 30.3848 48.2089 30.3763 48.0204C30.3677 47.8319 30.2992 47.6777 30.1707 47.5577C30.0421 47.4378 29.8772 47.3414 29.6759 47.2686C29.4745 47.1957 29.2453 47.1336 28.9883 47.0822C28.7313 47.0308 28.47 46.9751 28.2044 46.9152C27.9302 46.8552 27.6668 46.7824 27.414 46.6967C27.1613 46.611 26.9342 46.4953 26.7329 46.3497C26.5316 46.204 26.3709 46.0177 26.251 45.7907C26.131 45.5636 26.0711 45.283 26.0711 44.9489C26.0711 44.589 26.1589 44.287 26.3345 44.0429C26.5101 43.7987 26.7329 43.6016 27.0028 43.4517C27.2727 43.3018 27.5725 43.1968 27.9024 43.1368C28.2322 43.0769 28.5471 43.0469 28.847 43.0469C29.1897 43.0469 29.5174 43.0833 29.8301 43.1561C30.1428 43.2289 30.4255 43.3467 30.6783 43.5095C30.931 43.6723 31.1409 43.8844 31.308 44.1457C31.4751 44.407 31.58 44.7218 31.6229 45.0903H30.0935C30.025 44.739 29.8644 44.5034 29.6116 44.3834C29.3589 44.2635 29.0697 44.2035 28.7441 44.2035C28.6413 44.2035 28.5192 44.2121 28.3779 44.2292C28.2365 44.2463 28.1037 44.2785 27.9795 44.3256C27.8553 44.3727 27.7503 44.4413 27.6646 44.5312C27.579 44.6212 27.5361 44.739 27.5361 44.8846C27.5361 45.0645 27.5982 45.2102 27.7225 45.3216C27.8467 45.433 28.0095 45.5251 28.2108 45.5979C28.4122 45.6707 28.6413 45.7328 28.8984 45.7842C29.1554 45.8356 29.421 45.8913 29.6952 45.9513C29.9607 46.0113 30.2221 46.0841 30.4791 46.1698C30.7361 46.2554 30.9653 46.3711 31.1666 46.5168C31.368 46.6624 31.5308 46.8466 31.655 47.0694C31.7792 47.2921 31.8413 47.5663 31.8413 47.8919C31.8413 48.286 31.7514 48.6201 31.5715 48.8943C31.3915 49.1684 31.1581 49.3912 30.8711 49.5625C30.584 49.7339 30.2649 49.8581 29.9136 49.9352ZM34.4582 43.2266V49.8707H32.9932V43.2266H34.4582ZM38.8319 50.0511C39.2518 50.0511 39.6502 49.9676 40.0271 49.8005C40.4041 49.6335 40.6911 49.3572 40.8882 48.9716H40.9139V49.8712H42.3018V40.6953H40.8368V44.0881H40.8111C40.6997 43.9082 40.5605 43.7518 40.3934 43.619C40.2263 43.4862 40.0443 43.377 39.8472 43.2913C39.6502 43.2056 39.4445 43.1435 39.2303 43.1049C39.0161 43.0664 38.8062 43.0471 38.6006 43.0471C38.2408 43.0471 37.8874 43.1135 37.5404 43.2463C37.1934 43.3791 36.885 43.5869 36.6151 43.8696C36.3452 44.1523 36.1289 44.5122 35.9661 44.9491C35.8033 45.3861 35.7219 45.9044 35.7219 46.5041C35.7219 47.0011 35.7862 47.4659 35.9147 47.8985C36.0432 48.3312 36.236 48.706 36.493 49.023C36.75 49.34 37.0734 49.5906 37.4633 49.7748C37.8531 49.959 38.3093 50.0511 38.8319 50.0511ZM38.2027 48.7015C38.434 48.83 38.7082 48.8943 39.0252 48.8943C39.3508 48.8943 39.6314 48.8257 39.867 48.6886C40.1026 48.5516 40.2954 48.3716 40.4453 48.1489C40.5952 47.9261 40.7066 47.6755 40.7794 47.3971C40.8522 47.1186 40.8887 46.8338 40.8887 46.5425C40.8887 45.8056 40.7237 45.2316 40.3939 44.8204C40.064 44.4091 39.6164 44.2035 39.0509 44.2035C38.7082 44.2035 38.419 44.2742 38.1834 44.4155C37.9478 44.5569 37.7551 44.7411 37.6051 44.9682C37.4552 45.1952 37.3481 45.4522 37.2838 45.7392C37.2196 46.0263 37.1875 46.3197 37.1875 46.6196C37.1875 46.9023 37.2239 47.1807 37.2967 47.4549C37.3695 47.7291 37.4809 47.9732 37.6308 48.1874C37.7808 48.4016 37.9714 48.573 38.2027 48.7015ZM46.8334 50.0509C47.1933 50.0509 47.5381 50.0016 47.868 49.9031C48.1978 49.8046 48.4955 49.6589 48.7611 49.4662C49.0267 49.2734 49.2538 49.0356 49.4423 48.7529C49.6308 48.4702 49.7636 48.1489 49.8407 47.789H48.4527C48.3242 48.1575 48.1293 48.4338 47.868 48.618C47.6067 48.8022 47.2618 48.8943 46.8334 48.8943C46.525 48.8943 46.2594 48.8407 46.0366 48.7336C45.8139 48.6265 45.6297 48.483 45.484 48.3031C45.3384 48.1232 45.2292 47.9133 45.1563 47.6734C45.0835 47.4335 45.0471 47.185 45.0471 46.928H49.9306C49.9906 46.4311 49.9585 45.9492 49.8342 45.4822C49.71 45.0153 49.5108 44.6019 49.2366 44.2421C48.9625 43.8822 48.6176 43.5931 48.2021 43.3746C47.7866 43.1561 47.3175 43.0469 46.7949 43.0469C46.298 43.0469 45.8524 43.1411 45.4583 43.3296C45.0642 43.5181 44.7279 43.773 44.4495 44.0943C44.1711 44.4156 43.9569 44.7882 43.8069 45.2123C43.657 45.6364 43.582 46.0841 43.582 46.5553C43.582 47.0694 43.6527 47.5406 43.7941 47.969C43.9354 48.3973 44.1454 48.7658 44.4238 49.0742C44.7022 49.3826 45.0428 49.6225 45.4455 49.7939C45.8482 49.9652 46.3108 50.0509 46.8334 50.0509ZM45.0462 45.9641H48.4646C48.4475 45.7328 48.3982 45.5101 48.3168 45.2959C48.2354 45.0817 48.1241 44.8953 47.9827 44.7368C47.8413 44.5783 47.67 44.4498 47.4686 44.3513C47.2673 44.2528 47.0424 44.2035 46.7939 44.2035C46.5369 44.2035 46.3035 44.2485 46.0935 44.3384C45.8836 44.4284 45.7037 44.5526 45.5538 44.7111C45.4039 44.8696 45.2839 45.056 45.1939 45.2702C45.104 45.4844 45.0547 45.7157 45.0462 45.9641ZM52.5002 49.871V45.9513C52.5002 45.7114 52.5388 45.4844 52.6159 45.2702C52.693 45.056 52.8001 44.8696 52.9372 44.7111C53.0742 44.5526 53.2413 44.4284 53.4384 44.3384C53.6354 44.2485 53.8582 44.2035 54.1066 44.2035C54.5436 44.2035 54.8649 44.3213 55.0705 44.5569C55.2761 44.7925 55.3875 45.1716 55.4046 45.6943V49.871H56.8697V45.3087C56.8697 44.5548 56.6598 43.9893 56.24 43.6123C55.8202 43.2354 55.2461 43.0469 54.5179 43.0469C54.0809 43.0469 53.6825 43.1518 53.3227 43.3617C52.9629 43.5716 52.6716 43.8608 52.4488 44.2292L52.4231 44.2035V43.2268H51.0352V49.871H52.5002ZM61.172 49.9097C61.0178 49.9268 60.8593 49.9354 60.6965 49.9354C60.3109 49.9354 60.0004 49.8989 59.7647 49.8261C59.5291 49.7533 59.3449 49.6462 59.2121 49.5048C59.0793 49.3635 58.9894 49.1857 58.9423 48.9715C58.8951 48.7573 58.8673 48.5131 58.8587 48.239V44.3193H57.7535V43.2269H58.8587V41.235H60.3238V43.2269H61.6475V44.3193H60.3238V47.8663C60.3238 48.0205 60.3302 48.1533 60.3431 48.2647C60.3559 48.3761 60.3859 48.4703 60.433 48.5474C60.4801 48.6245 60.5508 48.6823 60.6451 48.7209C60.7393 48.7595 60.8678 48.7787 61.0306 48.7787C61.1334 48.7787 61.2362 48.7766 61.339 48.7723C61.4419 48.768 61.5447 48.753 61.6475 48.7273V49.8583C61.4847 49.8754 61.3262 49.8925 61.172 49.9097Z" fill="#E6E7EA"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M24.1732 35.1469C22.8656 35.5474 21.3518 35.7477 19.6319 35.7477C18.2418 35.7477 16.8929 35.5768 15.5853 35.2352C14.2776 34.8936 13.1232 34.3576 12.1218 33.6272C11.1205 32.8968 10.3253 31.9661 9.73627 30.8352C9.14725 29.7043 8.86452 28.3613 8.88808 26.8063H14.26C14.26 27.6545 14.4131 28.3731 14.7194 28.9621C15.0257 29.5511 15.4321 30.0282 15.9387 30.3934C16.4452 30.7586 17.0402 31.0296 17.7234 31.2063C18.4067 31.383 19.1135 31.4714 19.8439 31.4714C20.3387 31.4714 20.8688 31.4301 21.4343 31.3477C21.9997 31.2652 22.5298 31.1062 23.0246 30.8706C23.5194 30.6349 23.9317 30.311 24.2616 29.8987C24.5914 29.4863 24.7564 28.9621 24.7564 28.326C24.7564 27.6427 24.5384 27.089 24.1025 26.6649C23.6667 26.2408 23.0953 25.8874 22.3885 25.6047C21.6817 25.322 20.8806 25.0746 19.9853 24.8625C19.09 24.6505 18.1829 24.4149 17.264 24.1557C16.3215 23.9201 15.4027 23.6315 14.5074 23.2898C13.612 22.9482 12.811 22.5064 12.1041 21.9645C11.3973 21.4226 10.826 20.7452 10.3901 19.9324C9.95421 19.1195 9.73627 18.1359 9.73627 16.9814C9.73627 15.6855 10.0131 14.5605 10.5668 13.6063C11.1205 12.6521 11.845 11.8569 12.7403 11.2207C13.6356 10.5846 14.6487 10.1134 15.7796 9.80709C16.9106 9.5008 18.0415 9.34766 19.1724 9.34766C20.4918 9.34766 21.7582 9.49491 22.9716 9.78942C24.185 10.0839 25.2629 10.561 26.2053 11.2207C27.0062 11.7814 27.6668 12.4739 28.1869 13.2982V9.94832H33.2054V23.4841L39.5315 16.9106H45.4688L38.5773 23.6254L46.2463 35.1821H40.1676L35.1491 27.0182L33.2054 28.8913V35.1821H28.1869V32.7948C27.9637 33.0311 27.7214 33.2555 27.46 33.4681C26.5764 34.1867 25.4809 34.7463 24.1732 35.1469ZM28.1869 22.9297C27.8461 22.5904 27.4801 22.2923 27.0889 22.0352C26.2643 21.4933 25.4337 21.0987 24.5973 20.8513C23.7609 20.6039 23.1071 20.4331 22.6359 20.3388C21.0573 19.9383 19.7791 19.6084 18.8013 19.3493C17.8236 19.0901 17.0578 18.8309 16.5041 18.5718C15.9505 18.3126 15.5794 18.0299 15.3909 17.7236C15.2024 17.4173 15.1082 17.0167 15.1082 16.522C15.1082 15.9801 15.226 15.5324 15.4616 15.179C15.6972 14.8256 15.9976 14.5311 16.3628 14.2954C16.728 14.0598 17.1344 13.8949 17.5821 13.8007C18.0297 13.7064 18.4774 13.6593 18.925 13.6593C19.6083 13.6593 20.2385 13.7182 20.8158 13.836C21.393 13.9538 21.9055 14.1541 22.3531 14.4368C22.8008 14.7195 23.1601 15.1083 23.4311 15.6031C23.702 16.0979 23.861 16.7222 23.9082 17.4762H28.1869V22.9297Z" fill="#E6E7EA"/>
</svg>
</a></div></div><div class="footer__right"><div class="footer__lists"><div class="footer__list"><div class="footer__list-title notes-title">Products</div><div class="footer__list-item notes link"><a href="./software.html">MixCR.Pro</a></div><div class="footer__list-item notes link"><a href="./kits.html">Kits</a></div><div class="footer__list-item notes link"><a href="./services.html">Service</a></div><div class="footer__list-item notes link"><a href="./customR&amp;D.html">Custom R&amp;D</a></div></div><div class="footer__list"><div class="footer__list-title notes-title">About</div><div class="footer__list-item notes link"><a href="./publications.html">Publications</a></div><div class="footer__list-item notes link"><a href="./about.html">Team</a></div></div><div class="footer__list"><div class="footer__list-title notes-title">For clients</div><div class="footer__list-item notes link"><a href="./contact.html">Contacts</a></div></div></div></div><div class="footer__copyright"><p class="notes">© 2016-2020 All rights reserved</p><a class="notes" href="#">Privacy policy</a></div></div></footer></div><script src="https://cdn.plot.ly/plotly-2.0.0-rc.2.min.js"></script><script src="js/bundle.min.js"></script></body></html>